diff --git a/code/torch_nn.py b/code/torch_nn.py
index 661d37b..e92291f 100644
--- a/code/torch_nn.py
+++ b/code/torch_nn.py
@@ -239,7 +239,7 @@ config = dict(
     architecture = "fcnn",
     train_sims = list(train_sims),
     test_sims = list(test_sims),
-    n_frames = 20,
+    n_frames = 30,
     n_sims = n_sims,
     hidden_sizes = [128, 256, 128],
     activation_func = ["Tanh", "Tanh", "ReLU"],
diff --git a/code/wandb/latest-run b/code/wandb/latest-run
index b739db3..e519631 120000
--- a/code/wandb/latest-run
+++ b/code/wandb/latest-run
@@ -1 +1 @@
-run-20220914_140127-yctf6fn6
\ No newline at end of file
+run-20220914_170152-2p5hroko
\ No newline at end of file
