diff --git a/code/torch_nn.py b/code/torch_nn.py
index db0b44b..3a9de32 100644
--- a/code/torch_nn.py
+++ b/code/torch_nn.py
@@ -250,7 +250,7 @@ if __name__ == "__main__":
         loss_type = "L1",
         loss_reduction_type = "mean",
         optimizer = "Adam",
-        data_type = "pos",
+        data_type = "pos_diff_start",
         architecture = "fcnn",
         train_sims = list(train_sims),
         test_sims = list(test_sims),
diff --git a/code/wandb/latest-run b/code/wandb/latest-run
index 0a7bc9c..c923f90 120000
--- a/code/wandb/latest-run
+++ b/code/wandb/latest-run
@@ -1 +1 @@
-run-20220920_133124-1633qjkf
\ No newline at end of file
+run-20220920_140829-36ylmu32
\ No newline at end of file
