diff --git a/code/convert.py b/code/convert.py
index ce2684e..b9f8bc3 100644
--- a/code/convert.py
+++ b/code/convert.py
@@ -106,29 +106,29 @@ def log_quat2pos(log_quat, start_pos):
 
     # print(len(log_quat.shape))
     if len(log_quat.shape) == 2:
-        print(log_quat[:, 1:4].shape)
+        # print(log_quat[:, 1:4].shape)
 
-        v_norm = torch.linalg.norm(log_quat[:, 1:4], dim=1)[:, None].repeat(1, 3)
-        print("vnorm", v_norm.shape)
+        v_norm = torch.linalg.norm(log_quat[:, 1:4], dim=1)
+        # print("vnorm", v_norm.shape)
 
 
-        vec = log_quat[:, 1:4] / v_norm
-        print("vec", vec.shape)
+        vec = torch.div(log_quat[:, 1:4].T, v_norm).T
+        # print("vec", vec.shape)
 
-        magn = torch.exp(log_quat[:, 0])[:, None].repeat(1, 3)
-        print("magn", magn.shape)
+        magn = torch.exp(log_quat[:, 0])
+        # print("magn", magn.shape)
 
-        vector = magn * torch.sin(v_norm) * vec
+        vector = torch.mul(torch.mul(magn, torch.sin(v_norm)), vec.T).T
+        # print("1", torch.mul(magn, torch.sin(v_norm)).shape)
         print("vector", vector.shape)
 
-
-        scalar = magn * torch.cos(v_norm)
-        print("scalar", scalar[:,0])
+        scalar = (magn * torch.cos(v_norm))[:, None]
+        print("scalar", scalar.shape)
 
         # scalar = scalar.repeat(vector.shape[0], 1)
         # print("scalar2", scalar.shape)
 
-        quat = torch.hstack((scalar[:, 0], vector))
+        quat = torch.hstack((scalar, vector))
 
         print("quat", quat.shape)
 
diff --git a/code/data/sim_0.pickle b/code/data/sim_0.pickle
index cfa2e68..0fdf2c5 100644
Binary files a/code/data/sim_0.pickle and b/code/data/sim_0.pickle differ
diff --git a/code/data/sim_1.pickle b/code/data/sim_1.pickle
index cd9f577..4eaf778 100644
Binary files a/code/data/sim_1.pickle and b/code/data/sim_1.pickle differ
diff --git a/code/data/sim_10.pickle b/code/data/sim_10.pickle
index ef1a83d..e79d53e 100644
Binary files a/code/data/sim_10.pickle and b/code/data/sim_10.pickle differ
diff --git a/code/data/sim_100.pickle b/code/data/sim_100.pickle
index 4a7d552..f594bd4 100644
Binary files a/code/data/sim_100.pickle and b/code/data/sim_100.pickle differ
diff --git a/code/data/sim_101.pickle b/code/data/sim_101.pickle
index c2ed4ed..79faa62 100644
Binary files a/code/data/sim_101.pickle and b/code/data/sim_101.pickle differ
diff --git a/code/data/sim_102.pickle b/code/data/sim_102.pickle
index 1f12cb5..b6253c1 100644
Binary files a/code/data/sim_102.pickle and b/code/data/sim_102.pickle differ
diff --git a/code/data/sim_103.pickle b/code/data/sim_103.pickle
index 9c7eebb..8a99875 100644
Binary files a/code/data/sim_103.pickle and b/code/data/sim_103.pickle differ
diff --git a/code/data/sim_104.pickle b/code/data/sim_104.pickle
index 77eb5ad..a0d4057 100644
Binary files a/code/data/sim_104.pickle and b/code/data/sim_104.pickle differ
diff --git a/code/data/sim_105.pickle b/code/data/sim_105.pickle
index 826ef2c..faa24eb 100644
Binary files a/code/data/sim_105.pickle and b/code/data/sim_105.pickle differ
diff --git a/code/data/sim_106.pickle b/code/data/sim_106.pickle
index 461ee75..9f992cc 100644
Binary files a/code/data/sim_106.pickle and b/code/data/sim_106.pickle differ
diff --git a/code/data/sim_107.pickle b/code/data/sim_107.pickle
index 8c14ebc..26f4682 100644
Binary files a/code/data/sim_107.pickle and b/code/data/sim_107.pickle differ
diff --git a/code/data/sim_108.pickle b/code/data/sim_108.pickle
index c7e1827..bb8f316 100644
Binary files a/code/data/sim_108.pickle and b/code/data/sim_108.pickle differ
diff --git a/code/data/sim_109.pickle b/code/data/sim_109.pickle
index 2d4a342..151ddae 100644
Binary files a/code/data/sim_109.pickle and b/code/data/sim_109.pickle differ
diff --git a/code/data/sim_11.pickle b/code/data/sim_11.pickle
index 6052e0d..40f5839 100644
Binary files a/code/data/sim_11.pickle and b/code/data/sim_11.pickle differ
diff --git a/code/data/sim_110.pickle b/code/data/sim_110.pickle
index 80b0025..03600b0 100644
Binary files a/code/data/sim_110.pickle and b/code/data/sim_110.pickle differ
diff --git a/code/data/sim_111.pickle b/code/data/sim_111.pickle
index e375ca4..15f345a 100644
Binary files a/code/data/sim_111.pickle and b/code/data/sim_111.pickle differ
diff --git a/code/data/sim_112.pickle b/code/data/sim_112.pickle
index de4c1c2..9b9c42e 100644
Binary files a/code/data/sim_112.pickle and b/code/data/sim_112.pickle differ
diff --git a/code/data/sim_113.pickle b/code/data/sim_113.pickle
index 071b7ac..1a91ea2 100644
Binary files a/code/data/sim_113.pickle and b/code/data/sim_113.pickle differ
diff --git a/code/data/sim_114.pickle b/code/data/sim_114.pickle
index 8892019..c4fb7eb 100644
Binary files a/code/data/sim_114.pickle and b/code/data/sim_114.pickle differ
diff --git a/code/data/sim_115.pickle b/code/data/sim_115.pickle
index 0aef311..8d66efa 100644
Binary files a/code/data/sim_115.pickle and b/code/data/sim_115.pickle differ
diff --git a/code/data/sim_116.pickle b/code/data/sim_116.pickle
index 84d7c08..1b678e6 100644
Binary files a/code/data/sim_116.pickle and b/code/data/sim_116.pickle differ
diff --git a/code/data/sim_117.pickle b/code/data/sim_117.pickle
index 8a138eb..0947243 100644
Binary files a/code/data/sim_117.pickle and b/code/data/sim_117.pickle differ
diff --git a/code/data/sim_118.pickle b/code/data/sim_118.pickle
index ae2d73c..ddb5b34 100644
Binary files a/code/data/sim_118.pickle and b/code/data/sim_118.pickle differ
diff --git a/code/data/sim_119.pickle b/code/data/sim_119.pickle
index ec21d0b..33d83ef 100644
Binary files a/code/data/sim_119.pickle and b/code/data/sim_119.pickle differ
diff --git a/code/data/sim_12.pickle b/code/data/sim_12.pickle
index f7a583f..95a008d 100644
Binary files a/code/data/sim_12.pickle and b/code/data/sim_12.pickle differ
diff --git a/code/data/sim_120.pickle b/code/data/sim_120.pickle
index eca9aeb..7fb34c1 100644
Binary files a/code/data/sim_120.pickle and b/code/data/sim_120.pickle differ
diff --git a/code/data/sim_121.pickle b/code/data/sim_121.pickle
index d8896f5..33d664a 100644
Binary files a/code/data/sim_121.pickle and b/code/data/sim_121.pickle differ
diff --git a/code/data/sim_122.pickle b/code/data/sim_122.pickle
index 5b34efd..eb839ea 100644
Binary files a/code/data/sim_122.pickle and b/code/data/sim_122.pickle differ
diff --git a/code/data/sim_123.pickle b/code/data/sim_123.pickle
index c79ee63..07bf863 100644
Binary files a/code/data/sim_123.pickle and b/code/data/sim_123.pickle differ
diff --git a/code/data/sim_124.pickle b/code/data/sim_124.pickle
index e4eb604..e239550 100644
Binary files a/code/data/sim_124.pickle and b/code/data/sim_124.pickle differ
diff --git a/code/data/sim_125.pickle b/code/data/sim_125.pickle
index b742d25..5ff9dc1 100644
Binary files a/code/data/sim_125.pickle and b/code/data/sim_125.pickle differ
diff --git a/code/data/sim_126.pickle b/code/data/sim_126.pickle
index a0f1a54..9866ad4 100644
Binary files a/code/data/sim_126.pickle and b/code/data/sim_126.pickle differ
diff --git a/code/data/sim_127.pickle b/code/data/sim_127.pickle
index 54794cf..5418b52 100644
Binary files a/code/data/sim_127.pickle and b/code/data/sim_127.pickle differ
diff --git a/code/data/sim_128.pickle b/code/data/sim_128.pickle
index 6d423c9..718cb62 100644
Binary files a/code/data/sim_128.pickle and b/code/data/sim_128.pickle differ
diff --git a/code/data/sim_129.pickle b/code/data/sim_129.pickle
index 07dc79b..0f60460 100644
Binary files a/code/data/sim_129.pickle and b/code/data/sim_129.pickle differ
diff --git a/code/data/sim_13.pickle b/code/data/sim_13.pickle
index b427be4..1570f73 100644
Binary files a/code/data/sim_13.pickle and b/code/data/sim_13.pickle differ
diff --git a/code/data/sim_130.pickle b/code/data/sim_130.pickle
index c63a337..85e0478 100644
Binary files a/code/data/sim_130.pickle and b/code/data/sim_130.pickle differ
diff --git a/code/data/sim_131.pickle b/code/data/sim_131.pickle
index c293be4..1f87634 100644
Binary files a/code/data/sim_131.pickle and b/code/data/sim_131.pickle differ
diff --git a/code/data/sim_132.pickle b/code/data/sim_132.pickle
index e0f0a6c..c7a3fc8 100644
Binary files a/code/data/sim_132.pickle and b/code/data/sim_132.pickle differ
diff --git a/code/data/sim_133.pickle b/code/data/sim_133.pickle
index bfefd3d..4fb0263 100644
Binary files a/code/data/sim_133.pickle and b/code/data/sim_133.pickle differ
diff --git a/code/data/sim_134.pickle b/code/data/sim_134.pickle
index c95f856..03e85a7 100644
Binary files a/code/data/sim_134.pickle and b/code/data/sim_134.pickle differ
diff --git a/code/data/sim_135.pickle b/code/data/sim_135.pickle
index 48b0fc5..d3aefe2 100644
Binary files a/code/data/sim_135.pickle and b/code/data/sim_135.pickle differ
diff --git a/code/data/sim_136.pickle b/code/data/sim_136.pickle
index 2ccaa5d..880317f 100644
Binary files a/code/data/sim_136.pickle and b/code/data/sim_136.pickle differ
diff --git a/code/data/sim_137.pickle b/code/data/sim_137.pickle
index d1df984..a1fae88 100644
Binary files a/code/data/sim_137.pickle and b/code/data/sim_137.pickle differ
diff --git a/code/data/sim_138.pickle b/code/data/sim_138.pickle
index 33257ab..ae129c7 100644
Binary files a/code/data/sim_138.pickle and b/code/data/sim_138.pickle differ
diff --git a/code/data/sim_139.pickle b/code/data/sim_139.pickle
index 87e74ae..4cb8e21 100644
Binary files a/code/data/sim_139.pickle and b/code/data/sim_139.pickle differ
diff --git a/code/data/sim_14.pickle b/code/data/sim_14.pickle
index 320363e..ac723cb 100644
Binary files a/code/data/sim_14.pickle and b/code/data/sim_14.pickle differ
diff --git a/code/data/sim_140.pickle b/code/data/sim_140.pickle
index 08cf072..e0baec1 100644
Binary files a/code/data/sim_140.pickle and b/code/data/sim_140.pickle differ
diff --git a/code/data/sim_141.pickle b/code/data/sim_141.pickle
index 430297b..ef1a738 100644
Binary files a/code/data/sim_141.pickle and b/code/data/sim_141.pickle differ
diff --git a/code/data/sim_142.pickle b/code/data/sim_142.pickle
index dae6072..c7c0d59 100644
Binary files a/code/data/sim_142.pickle and b/code/data/sim_142.pickle differ
diff --git a/code/data/sim_143.pickle b/code/data/sim_143.pickle
index 839184c..183d2ed 100644
Binary files a/code/data/sim_143.pickle and b/code/data/sim_143.pickle differ
diff --git a/code/data/sim_144.pickle b/code/data/sim_144.pickle
index 1d9a645..1a1b562 100644
Binary files a/code/data/sim_144.pickle and b/code/data/sim_144.pickle differ
diff --git a/code/data/sim_145.pickle b/code/data/sim_145.pickle
index 1955069..5a9cf7e 100644
Binary files a/code/data/sim_145.pickle and b/code/data/sim_145.pickle differ
diff --git a/code/data/sim_146.pickle b/code/data/sim_146.pickle
index 93d05f8..5df3192 100644
Binary files a/code/data/sim_146.pickle and b/code/data/sim_146.pickle differ
diff --git a/code/data/sim_147.pickle b/code/data/sim_147.pickle
index ff42135..db25cd5 100644
Binary files a/code/data/sim_147.pickle and b/code/data/sim_147.pickle differ
diff --git a/code/data/sim_148.pickle b/code/data/sim_148.pickle
index 62b07a9..02137d9 100644
Binary files a/code/data/sim_148.pickle and b/code/data/sim_148.pickle differ
diff --git a/code/data/sim_149.pickle b/code/data/sim_149.pickle
index 35ece66..eb906e7 100644
Binary files a/code/data/sim_149.pickle and b/code/data/sim_149.pickle differ
diff --git a/code/data/sim_15.pickle b/code/data/sim_15.pickle
index 1d76005..0d9a3a4 100644
Binary files a/code/data/sim_15.pickle and b/code/data/sim_15.pickle differ
diff --git a/code/data/sim_150.pickle b/code/data/sim_150.pickle
index 514ef94..70525f4 100644
Binary files a/code/data/sim_150.pickle and b/code/data/sim_150.pickle differ
diff --git a/code/data/sim_151.pickle b/code/data/sim_151.pickle
index 2dd70d7..85175ee 100644
Binary files a/code/data/sim_151.pickle and b/code/data/sim_151.pickle differ
diff --git a/code/data/sim_152.pickle b/code/data/sim_152.pickle
index e8873f3..9c8ad5a 100644
Binary files a/code/data/sim_152.pickle and b/code/data/sim_152.pickle differ
diff --git a/code/data/sim_153.pickle b/code/data/sim_153.pickle
index 70d820a..dfc753b 100644
Binary files a/code/data/sim_153.pickle and b/code/data/sim_153.pickle differ
diff --git a/code/data/sim_154.pickle b/code/data/sim_154.pickle
index b65c60c..1bf22c4 100644
Binary files a/code/data/sim_154.pickle and b/code/data/sim_154.pickle differ
diff --git a/code/data/sim_155.pickle b/code/data/sim_155.pickle
index 2c354f5..31ae895 100644
Binary files a/code/data/sim_155.pickle and b/code/data/sim_155.pickle differ
diff --git a/code/data/sim_156.pickle b/code/data/sim_156.pickle
index 8c89a3d..619be3d 100644
Binary files a/code/data/sim_156.pickle and b/code/data/sim_156.pickle differ
diff --git a/code/data/sim_157.pickle b/code/data/sim_157.pickle
index 07720a9..b8e74a8 100644
Binary files a/code/data/sim_157.pickle and b/code/data/sim_157.pickle differ
diff --git a/code/data/sim_158.pickle b/code/data/sim_158.pickle
index f220fba..125654c 100644
Binary files a/code/data/sim_158.pickle and b/code/data/sim_158.pickle differ
diff --git a/code/data/sim_159.pickle b/code/data/sim_159.pickle
index 9dff9ea..29a66bf 100644
Binary files a/code/data/sim_159.pickle and b/code/data/sim_159.pickle differ
diff --git a/code/data/sim_16.pickle b/code/data/sim_16.pickle
index 2c247c8..86457d6 100644
Binary files a/code/data/sim_16.pickle and b/code/data/sim_16.pickle differ
diff --git a/code/data/sim_160.pickle b/code/data/sim_160.pickle
index 10ba307..54d6e6c 100644
Binary files a/code/data/sim_160.pickle and b/code/data/sim_160.pickle differ
diff --git a/code/data/sim_161.pickle b/code/data/sim_161.pickle
index 969a570..e8bb3f3 100644
Binary files a/code/data/sim_161.pickle and b/code/data/sim_161.pickle differ
diff --git a/code/data/sim_162.pickle b/code/data/sim_162.pickle
index 729ec42..3eb42b9 100644
Binary files a/code/data/sim_162.pickle and b/code/data/sim_162.pickle differ
diff --git a/code/data/sim_163.pickle b/code/data/sim_163.pickle
index 8d42d5a..c6c8793 100644
Binary files a/code/data/sim_163.pickle and b/code/data/sim_163.pickle differ
diff --git a/code/data/sim_164.pickle b/code/data/sim_164.pickle
index 291aff5..9e136ae 100644
Binary files a/code/data/sim_164.pickle and b/code/data/sim_164.pickle differ
diff --git a/code/data/sim_165.pickle b/code/data/sim_165.pickle
index 4f6af35..170bd7e 100644
Binary files a/code/data/sim_165.pickle and b/code/data/sim_165.pickle differ
diff --git a/code/data/sim_166.pickle b/code/data/sim_166.pickle
index 11feec0..8d051f9 100644
Binary files a/code/data/sim_166.pickle and b/code/data/sim_166.pickle differ
diff --git a/code/data/sim_167.pickle b/code/data/sim_167.pickle
index bedd468..7e5f377 100644
Binary files a/code/data/sim_167.pickle and b/code/data/sim_167.pickle differ
diff --git a/code/data/sim_168.pickle b/code/data/sim_168.pickle
index e33d4f1..293d1e8 100644
Binary files a/code/data/sim_168.pickle and b/code/data/sim_168.pickle differ
diff --git a/code/data/sim_169.pickle b/code/data/sim_169.pickle
index 94e0eb0..4c490ba 100644
Binary files a/code/data/sim_169.pickle and b/code/data/sim_169.pickle differ
diff --git a/code/data/sim_17.pickle b/code/data/sim_17.pickle
index a05280d..3352a9e 100644
Binary files a/code/data/sim_17.pickle and b/code/data/sim_17.pickle differ
diff --git a/code/data/sim_170.pickle b/code/data/sim_170.pickle
index b272e46..334e2b7 100644
Binary files a/code/data/sim_170.pickle and b/code/data/sim_170.pickle differ
diff --git a/code/data/sim_171.pickle b/code/data/sim_171.pickle
index 3cd1372..42cad3b 100644
Binary files a/code/data/sim_171.pickle and b/code/data/sim_171.pickle differ
diff --git a/code/data/sim_172.pickle b/code/data/sim_172.pickle
index a3bcb66..bc134b1 100644
Binary files a/code/data/sim_172.pickle and b/code/data/sim_172.pickle differ
diff --git a/code/data/sim_173.pickle b/code/data/sim_173.pickle
index 5ec6013..a2e8952 100644
Binary files a/code/data/sim_173.pickle and b/code/data/sim_173.pickle differ
diff --git a/code/data/sim_174.pickle b/code/data/sim_174.pickle
index df95f2d..6bf1ec5 100644
Binary files a/code/data/sim_174.pickle and b/code/data/sim_174.pickle differ
diff --git a/code/data/sim_175.pickle b/code/data/sim_175.pickle
index 32c8051..743284f 100644
Binary files a/code/data/sim_175.pickle and b/code/data/sim_175.pickle differ
diff --git a/code/data/sim_176.pickle b/code/data/sim_176.pickle
index b6d4063..75ef534 100644
Binary files a/code/data/sim_176.pickle and b/code/data/sim_176.pickle differ
diff --git a/code/data/sim_177.pickle b/code/data/sim_177.pickle
index db77b20..c5e1c65 100644
Binary files a/code/data/sim_177.pickle and b/code/data/sim_177.pickle differ
diff --git a/code/data/sim_178.pickle b/code/data/sim_178.pickle
index 4bfe437..64ab5bb 100644
Binary files a/code/data/sim_178.pickle and b/code/data/sim_178.pickle differ
diff --git a/code/data/sim_179.pickle b/code/data/sim_179.pickle
index c16fd29..df4f8a6 100644
Binary files a/code/data/sim_179.pickle and b/code/data/sim_179.pickle differ
diff --git a/code/data/sim_18.pickle b/code/data/sim_18.pickle
index 6262a15..19299de 100644
Binary files a/code/data/sim_18.pickle and b/code/data/sim_18.pickle differ
diff --git a/code/data/sim_180.pickle b/code/data/sim_180.pickle
index 3130662..2b14d8d 100644
Binary files a/code/data/sim_180.pickle and b/code/data/sim_180.pickle differ
diff --git a/code/data/sim_181.pickle b/code/data/sim_181.pickle
index 902afbd..d183b3d 100644
Binary files a/code/data/sim_181.pickle and b/code/data/sim_181.pickle differ
diff --git a/code/data/sim_182.pickle b/code/data/sim_182.pickle
index ef0a8b7..3bb2277 100644
Binary files a/code/data/sim_182.pickle and b/code/data/sim_182.pickle differ
diff --git a/code/data/sim_183.pickle b/code/data/sim_183.pickle
index 206ff7d..2713307 100644
Binary files a/code/data/sim_183.pickle and b/code/data/sim_183.pickle differ
diff --git a/code/data/sim_184.pickle b/code/data/sim_184.pickle
index cf5e6da..b702e84 100644
Binary files a/code/data/sim_184.pickle and b/code/data/sim_184.pickle differ
diff --git a/code/data/sim_185.pickle b/code/data/sim_185.pickle
index 14a6a9f..a883bca 100644
Binary files a/code/data/sim_185.pickle and b/code/data/sim_185.pickle differ
diff --git a/code/data/sim_186.pickle b/code/data/sim_186.pickle
index 26c22b4..10f14dc 100644
Binary files a/code/data/sim_186.pickle and b/code/data/sim_186.pickle differ
diff --git a/code/data/sim_187.pickle b/code/data/sim_187.pickle
index 620f95b..7528a9b 100644
Binary files a/code/data/sim_187.pickle and b/code/data/sim_187.pickle differ
diff --git a/code/data/sim_188.pickle b/code/data/sim_188.pickle
index 0830640..5091c15 100644
Binary files a/code/data/sim_188.pickle and b/code/data/sim_188.pickle differ
diff --git a/code/data/sim_189.pickle b/code/data/sim_189.pickle
index 4207b08..beefdc4 100644
Binary files a/code/data/sim_189.pickle and b/code/data/sim_189.pickle differ
diff --git a/code/data/sim_19.pickle b/code/data/sim_19.pickle
index 0234106..04e0f53 100644
Binary files a/code/data/sim_19.pickle and b/code/data/sim_19.pickle differ
diff --git a/code/data/sim_190.pickle b/code/data/sim_190.pickle
index 3185e7c..b3deb8b 100644
Binary files a/code/data/sim_190.pickle and b/code/data/sim_190.pickle differ
diff --git a/code/data/sim_191.pickle b/code/data/sim_191.pickle
index b2b0e7e..bed2bd9 100644
Binary files a/code/data/sim_191.pickle and b/code/data/sim_191.pickle differ
diff --git a/code/data/sim_192.pickle b/code/data/sim_192.pickle
index f93f03b..d23b9b7 100644
Binary files a/code/data/sim_192.pickle and b/code/data/sim_192.pickle differ
diff --git a/code/data/sim_193.pickle b/code/data/sim_193.pickle
index 332ed85..1cd1fb0 100644
Binary files a/code/data/sim_193.pickle and b/code/data/sim_193.pickle differ
diff --git a/code/data/sim_194.pickle b/code/data/sim_194.pickle
index 544e0c9..2886d5e 100644
Binary files a/code/data/sim_194.pickle and b/code/data/sim_194.pickle differ
diff --git a/code/data/sim_195.pickle b/code/data/sim_195.pickle
index 088b0c9..7b878ad 100644
Binary files a/code/data/sim_195.pickle and b/code/data/sim_195.pickle differ
diff --git a/code/data/sim_196.pickle b/code/data/sim_196.pickle
index dcdd50a..9dd22be 100644
Binary files a/code/data/sim_196.pickle and b/code/data/sim_196.pickle differ
diff --git a/code/data/sim_197.pickle b/code/data/sim_197.pickle
index 255273a..d1b91ee 100644
Binary files a/code/data/sim_197.pickle and b/code/data/sim_197.pickle differ
diff --git a/code/data/sim_198.pickle b/code/data/sim_198.pickle
index f3e9605..f3cc2d6 100644
Binary files a/code/data/sim_198.pickle and b/code/data/sim_198.pickle differ
diff --git a/code/data/sim_199.pickle b/code/data/sim_199.pickle
index 173cc3f..62d3c26 100644
Binary files a/code/data/sim_199.pickle and b/code/data/sim_199.pickle differ
diff --git a/code/data/sim_2.pickle b/code/data/sim_2.pickle
index ec831d8..7495f1e 100644
Binary files a/code/data/sim_2.pickle and b/code/data/sim_2.pickle differ
diff --git a/code/data/sim_20.pickle b/code/data/sim_20.pickle
index d80d979..8deb5e1 100644
Binary files a/code/data/sim_20.pickle and b/code/data/sim_20.pickle differ
diff --git a/code/data/sim_200.pickle b/code/data/sim_200.pickle
index 2941edd..2f619e5 100644
Binary files a/code/data/sim_200.pickle and b/code/data/sim_200.pickle differ
diff --git a/code/data/sim_201.pickle b/code/data/sim_201.pickle
index d69a188..8ae4dff 100644
Binary files a/code/data/sim_201.pickle and b/code/data/sim_201.pickle differ
diff --git a/code/data/sim_202.pickle b/code/data/sim_202.pickle
index 24ea601..1a92244 100644
Binary files a/code/data/sim_202.pickle and b/code/data/sim_202.pickle differ
diff --git a/code/data/sim_203.pickle b/code/data/sim_203.pickle
index 5da71cc..346896b 100644
Binary files a/code/data/sim_203.pickle and b/code/data/sim_203.pickle differ
diff --git a/code/data/sim_204.pickle b/code/data/sim_204.pickle
index 619ecce..da51013 100644
Binary files a/code/data/sim_204.pickle and b/code/data/sim_204.pickle differ
diff --git a/code/data/sim_205.pickle b/code/data/sim_205.pickle
index 5cb6c55..1922171 100644
Binary files a/code/data/sim_205.pickle and b/code/data/sim_205.pickle differ
diff --git a/code/data/sim_206.pickle b/code/data/sim_206.pickle
index 513e531..d72839b 100644
Binary files a/code/data/sim_206.pickle and b/code/data/sim_206.pickle differ
diff --git a/code/data/sim_207.pickle b/code/data/sim_207.pickle
index ff40300..da03e3f 100644
Binary files a/code/data/sim_207.pickle and b/code/data/sim_207.pickle differ
diff --git a/code/data/sim_208.pickle b/code/data/sim_208.pickle
index 099a403..1a312d5 100644
Binary files a/code/data/sim_208.pickle and b/code/data/sim_208.pickle differ
diff --git a/code/data/sim_209.pickle b/code/data/sim_209.pickle
index 3e6a456..d7d0bf3 100644
Binary files a/code/data/sim_209.pickle and b/code/data/sim_209.pickle differ
diff --git a/code/data/sim_21.pickle b/code/data/sim_21.pickle
index 2b01d48..fbfbb0b 100644
Binary files a/code/data/sim_21.pickle and b/code/data/sim_21.pickle differ
diff --git a/code/data/sim_210.pickle b/code/data/sim_210.pickle
index a20514d..ff2ef95 100644
Binary files a/code/data/sim_210.pickle and b/code/data/sim_210.pickle differ
diff --git a/code/data/sim_211.pickle b/code/data/sim_211.pickle
index dc78711..df59e09 100644
Binary files a/code/data/sim_211.pickle and b/code/data/sim_211.pickle differ
diff --git a/code/data/sim_212.pickle b/code/data/sim_212.pickle
index 718bbea..0b40820 100644
Binary files a/code/data/sim_212.pickle and b/code/data/sim_212.pickle differ
diff --git a/code/data/sim_213.pickle b/code/data/sim_213.pickle
index 295348f..b7c7a3b 100644
Binary files a/code/data/sim_213.pickle and b/code/data/sim_213.pickle differ
diff --git a/code/data/sim_214.pickle b/code/data/sim_214.pickle
index 970bdf4..a8e9aa3 100644
Binary files a/code/data/sim_214.pickle and b/code/data/sim_214.pickle differ
diff --git a/code/data/sim_215.pickle b/code/data/sim_215.pickle
index ff3a180..7dc1be8 100644
Binary files a/code/data/sim_215.pickle and b/code/data/sim_215.pickle differ
diff --git a/code/data/sim_216.pickle b/code/data/sim_216.pickle
index 565d40c..6d2b2ae 100644
Binary files a/code/data/sim_216.pickle and b/code/data/sim_216.pickle differ
diff --git a/code/data/sim_217.pickle b/code/data/sim_217.pickle
index 8179fc0..cc95493 100644
Binary files a/code/data/sim_217.pickle and b/code/data/sim_217.pickle differ
diff --git a/code/data/sim_218.pickle b/code/data/sim_218.pickle
index f573b37..8360102 100644
Binary files a/code/data/sim_218.pickle and b/code/data/sim_218.pickle differ
diff --git a/code/data/sim_219.pickle b/code/data/sim_219.pickle
index 825ae23..1d6c8b0 100644
Binary files a/code/data/sim_219.pickle and b/code/data/sim_219.pickle differ
diff --git a/code/data/sim_22.pickle b/code/data/sim_22.pickle
index 3a83c8f..f82705e 100644
Binary files a/code/data/sim_22.pickle and b/code/data/sim_22.pickle differ
diff --git a/code/data/sim_220.pickle b/code/data/sim_220.pickle
index 22450d5..bd9bc37 100644
Binary files a/code/data/sim_220.pickle and b/code/data/sim_220.pickle differ
diff --git a/code/data/sim_221.pickle b/code/data/sim_221.pickle
index 13f8921..c449da0 100644
Binary files a/code/data/sim_221.pickle and b/code/data/sim_221.pickle differ
diff --git a/code/data/sim_222.pickle b/code/data/sim_222.pickle
index c55a98a..aaf508a 100644
Binary files a/code/data/sim_222.pickle and b/code/data/sim_222.pickle differ
diff --git a/code/data/sim_223.pickle b/code/data/sim_223.pickle
index 5bd1941..25de721 100644
Binary files a/code/data/sim_223.pickle and b/code/data/sim_223.pickle differ
diff --git a/code/data/sim_224.pickle b/code/data/sim_224.pickle
index b3c875a..1c21c83 100644
Binary files a/code/data/sim_224.pickle and b/code/data/sim_224.pickle differ
diff --git a/code/data/sim_225.pickle b/code/data/sim_225.pickle
index 4ce0076..8df5427 100644
Binary files a/code/data/sim_225.pickle and b/code/data/sim_225.pickle differ
diff --git a/code/data/sim_226.pickle b/code/data/sim_226.pickle
index e043d05..0443269 100644
Binary files a/code/data/sim_226.pickle and b/code/data/sim_226.pickle differ
diff --git a/code/data/sim_227.pickle b/code/data/sim_227.pickle
index ab3600d..3e3953e 100644
Binary files a/code/data/sim_227.pickle and b/code/data/sim_227.pickle differ
diff --git a/code/data/sim_228.pickle b/code/data/sim_228.pickle
index 59c1943..0bbf797 100644
Binary files a/code/data/sim_228.pickle and b/code/data/sim_228.pickle differ
diff --git a/code/data/sim_229.pickle b/code/data/sim_229.pickle
index 96669f2..8785f6f 100644
Binary files a/code/data/sim_229.pickle and b/code/data/sim_229.pickle differ
diff --git a/code/data/sim_23.pickle b/code/data/sim_23.pickle
index a3585e0..d6f4eb7 100644
Binary files a/code/data/sim_23.pickle and b/code/data/sim_23.pickle differ
diff --git a/code/data/sim_230.pickle b/code/data/sim_230.pickle
index 039ef05..65f545c 100644
Binary files a/code/data/sim_230.pickle and b/code/data/sim_230.pickle differ
diff --git a/code/data/sim_231.pickle b/code/data/sim_231.pickle
index eafe3f1..b9c8097 100644
Binary files a/code/data/sim_231.pickle and b/code/data/sim_231.pickle differ
diff --git a/code/data/sim_232.pickle b/code/data/sim_232.pickle
index 1ead84f..788b541 100644
Binary files a/code/data/sim_232.pickle and b/code/data/sim_232.pickle differ
diff --git a/code/data/sim_233.pickle b/code/data/sim_233.pickle
index bdec2d7..6cb44af 100644
Binary files a/code/data/sim_233.pickle and b/code/data/sim_233.pickle differ
diff --git a/code/data/sim_234.pickle b/code/data/sim_234.pickle
index 1e65c16..a5f4195 100644
Binary files a/code/data/sim_234.pickle and b/code/data/sim_234.pickle differ
diff --git a/code/data/sim_235.pickle b/code/data/sim_235.pickle
index 4392c01..293f997 100644
Binary files a/code/data/sim_235.pickle and b/code/data/sim_235.pickle differ
diff --git a/code/data/sim_236.pickle b/code/data/sim_236.pickle
index 192a74e..16a1658 100644
Binary files a/code/data/sim_236.pickle and b/code/data/sim_236.pickle differ
diff --git a/code/data/sim_237.pickle b/code/data/sim_237.pickle
index 4641845..c56ba19 100644
Binary files a/code/data/sim_237.pickle and b/code/data/sim_237.pickle differ
diff --git a/code/data/sim_238.pickle b/code/data/sim_238.pickle
index 280c2e8..b1c00d8 100644
Binary files a/code/data/sim_238.pickle and b/code/data/sim_238.pickle differ
diff --git a/code/data/sim_239.pickle b/code/data/sim_239.pickle
index 3f6499f..fc1546d 100644
Binary files a/code/data/sim_239.pickle and b/code/data/sim_239.pickle differ
diff --git a/code/data/sim_24.pickle b/code/data/sim_24.pickle
index 98eb26e..41d4879 100644
Binary files a/code/data/sim_24.pickle and b/code/data/sim_24.pickle differ
diff --git a/code/data/sim_240.pickle b/code/data/sim_240.pickle
index d6b5003..25e09ad 100644
Binary files a/code/data/sim_240.pickle and b/code/data/sim_240.pickle differ
diff --git a/code/data/sim_241.pickle b/code/data/sim_241.pickle
index f3dda2e..953c30e 100644
Binary files a/code/data/sim_241.pickle and b/code/data/sim_241.pickle differ
diff --git a/code/data/sim_242.pickle b/code/data/sim_242.pickle
index 5038707..06c525d 100644
Binary files a/code/data/sim_242.pickle and b/code/data/sim_242.pickle differ
diff --git a/code/data/sim_243.pickle b/code/data/sim_243.pickle
index f9c1cde..a8c3ec3 100644
Binary files a/code/data/sim_243.pickle and b/code/data/sim_243.pickle differ
diff --git a/code/data/sim_244.pickle b/code/data/sim_244.pickle
index 9f71fa4..1265d52 100644
Binary files a/code/data/sim_244.pickle and b/code/data/sim_244.pickle differ
diff --git a/code/data/sim_245.pickle b/code/data/sim_245.pickle
index d26cf18..c1f57a7 100644
Binary files a/code/data/sim_245.pickle and b/code/data/sim_245.pickle differ
diff --git a/code/data/sim_246.pickle b/code/data/sim_246.pickle
index ae2e7ab..e43dbed 100644
Binary files a/code/data/sim_246.pickle and b/code/data/sim_246.pickle differ
diff --git a/code/data/sim_247.pickle b/code/data/sim_247.pickle
index a566d3b..86061d3 100644
Binary files a/code/data/sim_247.pickle and b/code/data/sim_247.pickle differ
diff --git a/code/data/sim_248.pickle b/code/data/sim_248.pickle
index 438d1bc..b691e57 100644
Binary files a/code/data/sim_248.pickle and b/code/data/sim_248.pickle differ
diff --git a/code/data/sim_249.pickle b/code/data/sim_249.pickle
index f2d1802..160086b 100644
Binary files a/code/data/sim_249.pickle and b/code/data/sim_249.pickle differ
diff --git a/code/data/sim_25.pickle b/code/data/sim_25.pickle
index f540b22..8269371 100644
Binary files a/code/data/sim_25.pickle and b/code/data/sim_25.pickle differ
diff --git a/code/data/sim_250.pickle b/code/data/sim_250.pickle
index d644fe0..ce3ca88 100644
Binary files a/code/data/sim_250.pickle and b/code/data/sim_250.pickle differ
diff --git a/code/data/sim_251.pickle b/code/data/sim_251.pickle
index 73f0db3..4aab5c8 100644
Binary files a/code/data/sim_251.pickle and b/code/data/sim_251.pickle differ
diff --git a/code/data/sim_252.pickle b/code/data/sim_252.pickle
index 7322b6b..c8de015 100644
Binary files a/code/data/sim_252.pickle and b/code/data/sim_252.pickle differ
diff --git a/code/data/sim_253.pickle b/code/data/sim_253.pickle
index 4955208..913a120 100644
Binary files a/code/data/sim_253.pickle and b/code/data/sim_253.pickle differ
diff --git a/code/data/sim_254.pickle b/code/data/sim_254.pickle
index a24c588..ffec859 100644
Binary files a/code/data/sim_254.pickle and b/code/data/sim_254.pickle differ
diff --git a/code/data/sim_255.pickle b/code/data/sim_255.pickle
index 2d36847..569b019 100644
Binary files a/code/data/sim_255.pickle and b/code/data/sim_255.pickle differ
diff --git a/code/data/sim_256.pickle b/code/data/sim_256.pickle
index b88e67f..6af343e 100644
Binary files a/code/data/sim_256.pickle and b/code/data/sim_256.pickle differ
diff --git a/code/data/sim_257.pickle b/code/data/sim_257.pickle
index 0cf2f1c..d084594 100644
Binary files a/code/data/sim_257.pickle and b/code/data/sim_257.pickle differ
diff --git a/code/data/sim_258.pickle b/code/data/sim_258.pickle
index 7dd00f6..e56ddb1 100644
Binary files a/code/data/sim_258.pickle and b/code/data/sim_258.pickle differ
diff --git a/code/data/sim_259.pickle b/code/data/sim_259.pickle
index 4c818f7..4f79c96 100644
Binary files a/code/data/sim_259.pickle and b/code/data/sim_259.pickle differ
diff --git a/code/data/sim_26.pickle b/code/data/sim_26.pickle
index dc5a383..30c4f73 100644
Binary files a/code/data/sim_26.pickle and b/code/data/sim_26.pickle differ
diff --git a/code/data/sim_260.pickle b/code/data/sim_260.pickle
index 8bbcae1..01087ee 100644
Binary files a/code/data/sim_260.pickle and b/code/data/sim_260.pickle differ
diff --git a/code/data/sim_261.pickle b/code/data/sim_261.pickle
index e6c06ea..7ca03c1 100644
Binary files a/code/data/sim_261.pickle and b/code/data/sim_261.pickle differ
diff --git a/code/data/sim_262.pickle b/code/data/sim_262.pickle
index f07c9ec..9b11b35 100644
Binary files a/code/data/sim_262.pickle and b/code/data/sim_262.pickle differ
diff --git a/code/data/sim_263.pickle b/code/data/sim_263.pickle
index 5f828ce..7087966 100644
Binary files a/code/data/sim_263.pickle and b/code/data/sim_263.pickle differ
diff --git a/code/data/sim_264.pickle b/code/data/sim_264.pickle
index 9ec663e..8939cc8 100644
Binary files a/code/data/sim_264.pickle and b/code/data/sim_264.pickle differ
diff --git a/code/data/sim_265.pickle b/code/data/sim_265.pickle
index db42d90..f3cfc0e 100644
Binary files a/code/data/sim_265.pickle and b/code/data/sim_265.pickle differ
diff --git a/code/data/sim_266.pickle b/code/data/sim_266.pickle
index 902a611..8cc4bf4 100644
Binary files a/code/data/sim_266.pickle and b/code/data/sim_266.pickle differ
diff --git a/code/data/sim_267.pickle b/code/data/sim_267.pickle
index 0b547e9..208edd4 100644
Binary files a/code/data/sim_267.pickle and b/code/data/sim_267.pickle differ
diff --git a/code/data/sim_268.pickle b/code/data/sim_268.pickle
index c9e285b..1961b65 100644
Binary files a/code/data/sim_268.pickle and b/code/data/sim_268.pickle differ
diff --git a/code/data/sim_269.pickle b/code/data/sim_269.pickle
index 3b4c83f..917785b 100644
Binary files a/code/data/sim_269.pickle and b/code/data/sim_269.pickle differ
diff --git a/code/data/sim_27.pickle b/code/data/sim_27.pickle
index e6c2d72..5545796 100644
Binary files a/code/data/sim_27.pickle and b/code/data/sim_27.pickle differ
diff --git a/code/data/sim_270.pickle b/code/data/sim_270.pickle
index 7c2234b..0c0ea00 100644
Binary files a/code/data/sim_270.pickle and b/code/data/sim_270.pickle differ
diff --git a/code/data/sim_271.pickle b/code/data/sim_271.pickle
index fb10a05..14a721b 100644
Binary files a/code/data/sim_271.pickle and b/code/data/sim_271.pickle differ
diff --git a/code/data/sim_272.pickle b/code/data/sim_272.pickle
index 353f66f..e2dcbcd 100644
Binary files a/code/data/sim_272.pickle and b/code/data/sim_272.pickle differ
diff --git a/code/data/sim_273.pickle b/code/data/sim_273.pickle
index 3725dc0..aeca6c9 100644
Binary files a/code/data/sim_273.pickle and b/code/data/sim_273.pickle differ
diff --git a/code/data/sim_274.pickle b/code/data/sim_274.pickle
index 9c3c477..46ba330 100644
Binary files a/code/data/sim_274.pickle and b/code/data/sim_274.pickle differ
diff --git a/code/data/sim_275.pickle b/code/data/sim_275.pickle
index 5c3791e..b43be01 100644
Binary files a/code/data/sim_275.pickle and b/code/data/sim_275.pickle differ
diff --git a/code/data/sim_276.pickle b/code/data/sim_276.pickle
index d5c2929..eda75ae 100644
Binary files a/code/data/sim_276.pickle and b/code/data/sim_276.pickle differ
diff --git a/code/data/sim_277.pickle b/code/data/sim_277.pickle
index a47eccf..2ecaa5a 100644
Binary files a/code/data/sim_277.pickle and b/code/data/sim_277.pickle differ
diff --git a/code/data/sim_278.pickle b/code/data/sim_278.pickle
index ab643ab..af8ebdb 100644
Binary files a/code/data/sim_278.pickle and b/code/data/sim_278.pickle differ
diff --git a/code/data/sim_279.pickle b/code/data/sim_279.pickle
index cb14231..39ab397 100644
Binary files a/code/data/sim_279.pickle and b/code/data/sim_279.pickle differ
diff --git a/code/data/sim_28.pickle b/code/data/sim_28.pickle
index 1b38c39..2d0d3e6 100644
Binary files a/code/data/sim_28.pickle and b/code/data/sim_28.pickle differ
diff --git a/code/data/sim_280.pickle b/code/data/sim_280.pickle
index 2295389..9a107e7 100644
Binary files a/code/data/sim_280.pickle and b/code/data/sim_280.pickle differ
diff --git a/code/data/sim_281.pickle b/code/data/sim_281.pickle
index c759d96..72610dd 100644
Binary files a/code/data/sim_281.pickle and b/code/data/sim_281.pickle differ
diff --git a/code/data/sim_282.pickle b/code/data/sim_282.pickle
index 91636d1..6c2bce9 100644
Binary files a/code/data/sim_282.pickle and b/code/data/sim_282.pickle differ
diff --git a/code/data/sim_283.pickle b/code/data/sim_283.pickle
index 1e0f434..016747d 100644
Binary files a/code/data/sim_283.pickle and b/code/data/sim_283.pickle differ
diff --git a/code/data/sim_284.pickle b/code/data/sim_284.pickle
index 160eae3..cbe2199 100644
Binary files a/code/data/sim_284.pickle and b/code/data/sim_284.pickle differ
diff --git a/code/data/sim_285.pickle b/code/data/sim_285.pickle
index 49da5b3..c42f45f 100644
Binary files a/code/data/sim_285.pickle and b/code/data/sim_285.pickle differ
diff --git a/code/data/sim_286.pickle b/code/data/sim_286.pickle
index f2f4602..1e22486 100644
Binary files a/code/data/sim_286.pickle and b/code/data/sim_286.pickle differ
diff --git a/code/data/sim_287.pickle b/code/data/sim_287.pickle
index 07c2ba6..81f31bb 100644
Binary files a/code/data/sim_287.pickle and b/code/data/sim_287.pickle differ
diff --git a/code/data/sim_288.pickle b/code/data/sim_288.pickle
index 0df21ac..a08f951 100644
Binary files a/code/data/sim_288.pickle and b/code/data/sim_288.pickle differ
diff --git a/code/data/sim_289.pickle b/code/data/sim_289.pickle
index 0e835e6..40c31d8 100644
Binary files a/code/data/sim_289.pickle and b/code/data/sim_289.pickle differ
diff --git a/code/data/sim_29.pickle b/code/data/sim_29.pickle
index 92ddb08..36c12e3 100644
Binary files a/code/data/sim_29.pickle and b/code/data/sim_29.pickle differ
diff --git a/code/data/sim_290.pickle b/code/data/sim_290.pickle
index 935964f..76adfcd 100644
Binary files a/code/data/sim_290.pickle and b/code/data/sim_290.pickle differ
diff --git a/code/data/sim_291.pickle b/code/data/sim_291.pickle
index ab3f2ca..68d526c 100644
Binary files a/code/data/sim_291.pickle and b/code/data/sim_291.pickle differ
diff --git a/code/data/sim_292.pickle b/code/data/sim_292.pickle
index ccfdc46..3ca8484 100644
Binary files a/code/data/sim_292.pickle and b/code/data/sim_292.pickle differ
diff --git a/code/data/sim_293.pickle b/code/data/sim_293.pickle
index 018cb02..3d4fdeb 100644
Binary files a/code/data/sim_293.pickle and b/code/data/sim_293.pickle differ
diff --git a/code/data/sim_294.pickle b/code/data/sim_294.pickle
index d60d1bb..d10f244 100644
Binary files a/code/data/sim_294.pickle and b/code/data/sim_294.pickle differ
diff --git a/code/data/sim_295.pickle b/code/data/sim_295.pickle
index 8574619..567a72f 100644
Binary files a/code/data/sim_295.pickle and b/code/data/sim_295.pickle differ
diff --git a/code/data/sim_296.pickle b/code/data/sim_296.pickle
index 099df62..54c0072 100644
Binary files a/code/data/sim_296.pickle and b/code/data/sim_296.pickle differ
diff --git a/code/data/sim_297.pickle b/code/data/sim_297.pickle
index b9feeac..c7b0e1a 100644
Binary files a/code/data/sim_297.pickle and b/code/data/sim_297.pickle differ
diff --git a/code/data/sim_298.pickle b/code/data/sim_298.pickle
index a6dde31..2b0bae4 100644
Binary files a/code/data/sim_298.pickle and b/code/data/sim_298.pickle differ
diff --git a/code/data/sim_299.pickle b/code/data/sim_299.pickle
index bfa47cd..d766b00 100644
Binary files a/code/data/sim_299.pickle and b/code/data/sim_299.pickle differ
diff --git a/code/data/sim_3.pickle b/code/data/sim_3.pickle
index 65a5bb9..0cd489a 100644
Binary files a/code/data/sim_3.pickle and b/code/data/sim_3.pickle differ
diff --git a/code/data/sim_30.pickle b/code/data/sim_30.pickle
index 18e8e7d..37347d7 100644
Binary files a/code/data/sim_30.pickle and b/code/data/sim_30.pickle differ
diff --git a/code/data/sim_300.pickle b/code/data/sim_300.pickle
index f5513f7..e842ce3 100644
Binary files a/code/data/sim_300.pickle and b/code/data/sim_300.pickle differ
diff --git a/code/data/sim_301.pickle b/code/data/sim_301.pickle
index bc434ed..eb72509 100644
Binary files a/code/data/sim_301.pickle and b/code/data/sim_301.pickle differ
diff --git a/code/data/sim_302.pickle b/code/data/sim_302.pickle
index 738e4db..3a4f064 100644
Binary files a/code/data/sim_302.pickle and b/code/data/sim_302.pickle differ
diff --git a/code/data/sim_303.pickle b/code/data/sim_303.pickle
index 46b44e3..854dfc8 100644
Binary files a/code/data/sim_303.pickle and b/code/data/sim_303.pickle differ
diff --git a/code/data/sim_304.pickle b/code/data/sim_304.pickle
index 13a724b..a7b4ce0 100644
Binary files a/code/data/sim_304.pickle and b/code/data/sim_304.pickle differ
diff --git a/code/data/sim_305.pickle b/code/data/sim_305.pickle
index 2c6cda8..b45653b 100644
Binary files a/code/data/sim_305.pickle and b/code/data/sim_305.pickle differ
diff --git a/code/data/sim_306.pickle b/code/data/sim_306.pickle
index 4e0202c..7f79485 100644
Binary files a/code/data/sim_306.pickle and b/code/data/sim_306.pickle differ
diff --git a/code/data/sim_307.pickle b/code/data/sim_307.pickle
index b3b1b7d..7d900f5 100644
Binary files a/code/data/sim_307.pickle and b/code/data/sim_307.pickle differ
diff --git a/code/data/sim_308.pickle b/code/data/sim_308.pickle
index ca9cdc6..2e0af99 100644
Binary files a/code/data/sim_308.pickle and b/code/data/sim_308.pickle differ
diff --git a/code/data/sim_309.pickle b/code/data/sim_309.pickle
index 82dfd43..2eb970b 100644
Binary files a/code/data/sim_309.pickle and b/code/data/sim_309.pickle differ
diff --git a/code/data/sim_31.pickle b/code/data/sim_31.pickle
index 56169a1..dc1b981 100644
Binary files a/code/data/sim_31.pickle and b/code/data/sim_31.pickle differ
diff --git a/code/data/sim_310.pickle b/code/data/sim_310.pickle
index 7352be5..88b2058 100644
Binary files a/code/data/sim_310.pickle and b/code/data/sim_310.pickle differ
diff --git a/code/data/sim_311.pickle b/code/data/sim_311.pickle
index b30a0ff..f66f9f3 100644
Binary files a/code/data/sim_311.pickle and b/code/data/sim_311.pickle differ
diff --git a/code/data/sim_312.pickle b/code/data/sim_312.pickle
index 89710e9..ad7681b 100644
Binary files a/code/data/sim_312.pickle and b/code/data/sim_312.pickle differ
diff --git a/code/data/sim_313.pickle b/code/data/sim_313.pickle
index 3d1a369..ea0705b 100644
Binary files a/code/data/sim_313.pickle and b/code/data/sim_313.pickle differ
diff --git a/code/data/sim_314.pickle b/code/data/sim_314.pickle
index 751edf7..77898a2 100644
Binary files a/code/data/sim_314.pickle and b/code/data/sim_314.pickle differ
diff --git a/code/data/sim_315.pickle b/code/data/sim_315.pickle
index 1fb8cad..f265535 100644
Binary files a/code/data/sim_315.pickle and b/code/data/sim_315.pickle differ
diff --git a/code/data/sim_316.pickle b/code/data/sim_316.pickle
index 54606b0..b85952c 100644
Binary files a/code/data/sim_316.pickle and b/code/data/sim_316.pickle differ
diff --git a/code/data/sim_317.pickle b/code/data/sim_317.pickle
index 877760f..fcb74cb 100644
Binary files a/code/data/sim_317.pickle and b/code/data/sim_317.pickle differ
diff --git a/code/data/sim_318.pickle b/code/data/sim_318.pickle
index 1171888..ef3db7f 100644
Binary files a/code/data/sim_318.pickle and b/code/data/sim_318.pickle differ
diff --git a/code/data/sim_319.pickle b/code/data/sim_319.pickle
index c562aa3..532c0ac 100644
Binary files a/code/data/sim_319.pickle and b/code/data/sim_319.pickle differ
diff --git a/code/data/sim_32.pickle b/code/data/sim_32.pickle
index 1cce6dc..b47ca07 100644
Binary files a/code/data/sim_32.pickle and b/code/data/sim_32.pickle differ
diff --git a/code/data/sim_320.pickle b/code/data/sim_320.pickle
index 1569780..91f14ba 100644
Binary files a/code/data/sim_320.pickle and b/code/data/sim_320.pickle differ
diff --git a/code/data/sim_321.pickle b/code/data/sim_321.pickle
index 68f65f2..b535330 100644
Binary files a/code/data/sim_321.pickle and b/code/data/sim_321.pickle differ
diff --git a/code/data/sim_322.pickle b/code/data/sim_322.pickle
index d21882f..6165855 100644
Binary files a/code/data/sim_322.pickle and b/code/data/sim_322.pickle differ
diff --git a/code/data/sim_323.pickle b/code/data/sim_323.pickle
index 7ec685a..229bf63 100644
Binary files a/code/data/sim_323.pickle and b/code/data/sim_323.pickle differ
diff --git a/code/data/sim_324.pickle b/code/data/sim_324.pickle
index 887fd11..755549c 100644
Binary files a/code/data/sim_324.pickle and b/code/data/sim_324.pickle differ
diff --git a/code/data/sim_325.pickle b/code/data/sim_325.pickle
index 92919ef..9763092 100644
Binary files a/code/data/sim_325.pickle and b/code/data/sim_325.pickle differ
diff --git a/code/data/sim_326.pickle b/code/data/sim_326.pickle
index a2fff5e..7d16b71 100644
Binary files a/code/data/sim_326.pickle and b/code/data/sim_326.pickle differ
diff --git a/code/data/sim_327.pickle b/code/data/sim_327.pickle
index 988cdce..c6e3bfb 100644
Binary files a/code/data/sim_327.pickle and b/code/data/sim_327.pickle differ
diff --git a/code/data/sim_328.pickle b/code/data/sim_328.pickle
index 2e79e68..b56033a 100644
Binary files a/code/data/sim_328.pickle and b/code/data/sim_328.pickle differ
diff --git a/code/data/sim_329.pickle b/code/data/sim_329.pickle
index 664ff46..a75900b 100644
Binary files a/code/data/sim_329.pickle and b/code/data/sim_329.pickle differ
diff --git a/code/data/sim_33.pickle b/code/data/sim_33.pickle
index 5129f26..a016505 100644
Binary files a/code/data/sim_33.pickle and b/code/data/sim_33.pickle differ
diff --git a/code/data/sim_330.pickle b/code/data/sim_330.pickle
index 86a7fc2..59289bd 100644
Binary files a/code/data/sim_330.pickle and b/code/data/sim_330.pickle differ
diff --git a/code/data/sim_331.pickle b/code/data/sim_331.pickle
index 1ae7874..3c33bb6 100644
Binary files a/code/data/sim_331.pickle and b/code/data/sim_331.pickle differ
diff --git a/code/data/sim_332.pickle b/code/data/sim_332.pickle
index 1f34f87..f1ffea4 100644
Binary files a/code/data/sim_332.pickle and b/code/data/sim_332.pickle differ
diff --git a/code/data/sim_333.pickle b/code/data/sim_333.pickle
index 91477c4..fd99a9f 100644
Binary files a/code/data/sim_333.pickle and b/code/data/sim_333.pickle differ
diff --git a/code/data/sim_334.pickle b/code/data/sim_334.pickle
index 1f9dcdb..617a0c9 100644
Binary files a/code/data/sim_334.pickle and b/code/data/sim_334.pickle differ
diff --git a/code/data/sim_335.pickle b/code/data/sim_335.pickle
index 03ee4d5..3a70b07 100644
Binary files a/code/data/sim_335.pickle and b/code/data/sim_335.pickle differ
diff --git a/code/data/sim_336.pickle b/code/data/sim_336.pickle
index c27568c..c2713c8 100644
Binary files a/code/data/sim_336.pickle and b/code/data/sim_336.pickle differ
diff --git a/code/data/sim_337.pickle b/code/data/sim_337.pickle
index 4b6c793..417d9e0 100644
Binary files a/code/data/sim_337.pickle and b/code/data/sim_337.pickle differ
diff --git a/code/data/sim_338.pickle b/code/data/sim_338.pickle
index 3e587d6..e262dc3 100644
Binary files a/code/data/sim_338.pickle and b/code/data/sim_338.pickle differ
diff --git a/code/data/sim_339.pickle b/code/data/sim_339.pickle
index 99ad124..666158d 100644
Binary files a/code/data/sim_339.pickle and b/code/data/sim_339.pickle differ
diff --git a/code/data/sim_34.pickle b/code/data/sim_34.pickle
index 9c56079..eb6b601 100644
Binary files a/code/data/sim_34.pickle and b/code/data/sim_34.pickle differ
diff --git a/code/data/sim_340.pickle b/code/data/sim_340.pickle
index 307e748..f908c55 100644
Binary files a/code/data/sim_340.pickle and b/code/data/sim_340.pickle differ
diff --git a/code/data/sim_341.pickle b/code/data/sim_341.pickle
index e2c462c..3e4055f 100644
Binary files a/code/data/sim_341.pickle and b/code/data/sim_341.pickle differ
diff --git a/code/data/sim_342.pickle b/code/data/sim_342.pickle
index 3242bf3..617ce78 100644
Binary files a/code/data/sim_342.pickle and b/code/data/sim_342.pickle differ
diff --git a/code/data/sim_343.pickle b/code/data/sim_343.pickle
index 3f1ad7d..44667b2 100644
Binary files a/code/data/sim_343.pickle and b/code/data/sim_343.pickle differ
diff --git a/code/data/sim_344.pickle b/code/data/sim_344.pickle
index 48c111d..81790ad 100644
Binary files a/code/data/sim_344.pickle and b/code/data/sim_344.pickle differ
diff --git a/code/data/sim_345.pickle b/code/data/sim_345.pickle
index 1368392..054b80d 100644
Binary files a/code/data/sim_345.pickle and b/code/data/sim_345.pickle differ
diff --git a/code/data/sim_346.pickle b/code/data/sim_346.pickle
index 779034d..b06168d 100644
Binary files a/code/data/sim_346.pickle and b/code/data/sim_346.pickle differ
diff --git a/code/data/sim_347.pickle b/code/data/sim_347.pickle
index e9b31f3..806944d 100644
Binary files a/code/data/sim_347.pickle and b/code/data/sim_347.pickle differ
diff --git a/code/data/sim_348.pickle b/code/data/sim_348.pickle
index 661c764..f6a4b92 100644
Binary files a/code/data/sim_348.pickle and b/code/data/sim_348.pickle differ
diff --git a/code/data/sim_349.pickle b/code/data/sim_349.pickle
index 38042a2..5bcef55 100644
Binary files a/code/data/sim_349.pickle and b/code/data/sim_349.pickle differ
diff --git a/code/data/sim_35.pickle b/code/data/sim_35.pickle
index b93834d..be90e52 100644
Binary files a/code/data/sim_35.pickle and b/code/data/sim_35.pickle differ
diff --git a/code/data/sim_350.pickle b/code/data/sim_350.pickle
index 6c1e2ca..3dc606d 100644
Binary files a/code/data/sim_350.pickle and b/code/data/sim_350.pickle differ
diff --git a/code/data/sim_351.pickle b/code/data/sim_351.pickle
index 417480e..7bc61e9 100644
Binary files a/code/data/sim_351.pickle and b/code/data/sim_351.pickle differ
diff --git a/code/data/sim_352.pickle b/code/data/sim_352.pickle
index 5307fa9..d65992e 100644
Binary files a/code/data/sim_352.pickle and b/code/data/sim_352.pickle differ
diff --git a/code/data/sim_353.pickle b/code/data/sim_353.pickle
index 8828c54..5626e7f 100644
Binary files a/code/data/sim_353.pickle and b/code/data/sim_353.pickle differ
diff --git a/code/data/sim_354.pickle b/code/data/sim_354.pickle
index 8b318e2..f3ae290 100644
Binary files a/code/data/sim_354.pickle and b/code/data/sim_354.pickle differ
diff --git a/code/data/sim_355.pickle b/code/data/sim_355.pickle
index f6deaea..851aa81 100644
Binary files a/code/data/sim_355.pickle and b/code/data/sim_355.pickle differ
diff --git a/code/data/sim_356.pickle b/code/data/sim_356.pickle
index fb34dfb..3584996 100644
Binary files a/code/data/sim_356.pickle and b/code/data/sim_356.pickle differ
diff --git a/code/data/sim_357.pickle b/code/data/sim_357.pickle
index 0738db0..e7c67cd 100644
Binary files a/code/data/sim_357.pickle and b/code/data/sim_357.pickle differ
diff --git a/code/data/sim_358.pickle b/code/data/sim_358.pickle
index 8cb9878..77b5f39 100644
Binary files a/code/data/sim_358.pickle and b/code/data/sim_358.pickle differ
diff --git a/code/data/sim_359.pickle b/code/data/sim_359.pickle
index 66b8adf..c2fa0c8 100644
Binary files a/code/data/sim_359.pickle and b/code/data/sim_359.pickle differ
diff --git a/code/data/sim_36.pickle b/code/data/sim_36.pickle
index bae3d8e..e1a789d 100644
Binary files a/code/data/sim_36.pickle and b/code/data/sim_36.pickle differ
diff --git a/code/data/sim_360.pickle b/code/data/sim_360.pickle
index 44b44a9..cefc69a 100644
Binary files a/code/data/sim_360.pickle and b/code/data/sim_360.pickle differ
diff --git a/code/data/sim_361.pickle b/code/data/sim_361.pickle
index 6c94867..1a668fa 100644
Binary files a/code/data/sim_361.pickle and b/code/data/sim_361.pickle differ
diff --git a/code/data/sim_362.pickle b/code/data/sim_362.pickle
index 5630470..053b66b 100644
Binary files a/code/data/sim_362.pickle and b/code/data/sim_362.pickle differ
diff --git a/code/data/sim_363.pickle b/code/data/sim_363.pickle
index 6e501c0..db5df12 100644
Binary files a/code/data/sim_363.pickle and b/code/data/sim_363.pickle differ
diff --git a/code/data/sim_364.pickle b/code/data/sim_364.pickle
index d80b0e1..9cd8db0 100644
Binary files a/code/data/sim_364.pickle and b/code/data/sim_364.pickle differ
diff --git a/code/data/sim_365.pickle b/code/data/sim_365.pickle
index 931a705..b3eb680 100644
Binary files a/code/data/sim_365.pickle and b/code/data/sim_365.pickle differ
diff --git a/code/data/sim_366.pickle b/code/data/sim_366.pickle
index 400c960..c5f2d10 100644
Binary files a/code/data/sim_366.pickle and b/code/data/sim_366.pickle differ
diff --git a/code/data/sim_367.pickle b/code/data/sim_367.pickle
index fb1472e..1f9e8b5 100644
Binary files a/code/data/sim_367.pickle and b/code/data/sim_367.pickle differ
diff --git a/code/data/sim_368.pickle b/code/data/sim_368.pickle
index 378f37c..51b6eb9 100644
Binary files a/code/data/sim_368.pickle and b/code/data/sim_368.pickle differ
diff --git a/code/data/sim_369.pickle b/code/data/sim_369.pickle
index 6198cd7..4879794 100644
Binary files a/code/data/sim_369.pickle and b/code/data/sim_369.pickle differ
diff --git a/code/data/sim_37.pickle b/code/data/sim_37.pickle
index 96b3617..b7b25ed 100644
Binary files a/code/data/sim_37.pickle and b/code/data/sim_37.pickle differ
diff --git a/code/data/sim_370.pickle b/code/data/sim_370.pickle
index 1a5f233..6a25ef2 100644
Binary files a/code/data/sim_370.pickle and b/code/data/sim_370.pickle differ
diff --git a/code/data/sim_371.pickle b/code/data/sim_371.pickle
index 86aa934..b08fc1c 100644
Binary files a/code/data/sim_371.pickle and b/code/data/sim_371.pickle differ
diff --git a/code/data/sim_372.pickle b/code/data/sim_372.pickle
index fc1819d..be46ce9 100644
Binary files a/code/data/sim_372.pickle and b/code/data/sim_372.pickle differ
diff --git a/code/data/sim_373.pickle b/code/data/sim_373.pickle
index ebdbdf8..a55c1da 100644
Binary files a/code/data/sim_373.pickle and b/code/data/sim_373.pickle differ
diff --git a/code/data/sim_374.pickle b/code/data/sim_374.pickle
index 6a6c312..22edab5 100644
Binary files a/code/data/sim_374.pickle and b/code/data/sim_374.pickle differ
diff --git a/code/data/sim_375.pickle b/code/data/sim_375.pickle
index d6f8b98..647acac 100644
Binary files a/code/data/sim_375.pickle and b/code/data/sim_375.pickle differ
diff --git a/code/data/sim_376.pickle b/code/data/sim_376.pickle
index cdab4d8..bbb24eb 100644
Binary files a/code/data/sim_376.pickle and b/code/data/sim_376.pickle differ
diff --git a/code/data/sim_377.pickle b/code/data/sim_377.pickle
index ba29ee8..0484e3f 100644
Binary files a/code/data/sim_377.pickle and b/code/data/sim_377.pickle differ
diff --git a/code/data/sim_378.pickle b/code/data/sim_378.pickle
index 73ea0cb..bb00f7a 100644
Binary files a/code/data/sim_378.pickle and b/code/data/sim_378.pickle differ
diff --git a/code/data/sim_379.pickle b/code/data/sim_379.pickle
index 4635c53..0f96743 100644
Binary files a/code/data/sim_379.pickle and b/code/data/sim_379.pickle differ
diff --git a/code/data/sim_38.pickle b/code/data/sim_38.pickle
index 141e43c..94f3726 100644
Binary files a/code/data/sim_38.pickle and b/code/data/sim_38.pickle differ
diff --git a/code/data/sim_380.pickle b/code/data/sim_380.pickle
index 68b7725..9f8cf52 100644
Binary files a/code/data/sim_380.pickle and b/code/data/sim_380.pickle differ
diff --git a/code/data/sim_381.pickle b/code/data/sim_381.pickle
index 71b8f4d..dd356ba 100644
Binary files a/code/data/sim_381.pickle and b/code/data/sim_381.pickle differ
diff --git a/code/data/sim_382.pickle b/code/data/sim_382.pickle
index 771c0b4..b1cb8ae 100644
Binary files a/code/data/sim_382.pickle and b/code/data/sim_382.pickle differ
diff --git a/code/data/sim_383.pickle b/code/data/sim_383.pickle
index 2e666a2..8e827a2 100644
Binary files a/code/data/sim_383.pickle and b/code/data/sim_383.pickle differ
diff --git a/code/data/sim_384.pickle b/code/data/sim_384.pickle
index 119cbce..4996dc1 100644
Binary files a/code/data/sim_384.pickle and b/code/data/sim_384.pickle differ
diff --git a/code/data/sim_385.pickle b/code/data/sim_385.pickle
index e234956..ede323f 100644
Binary files a/code/data/sim_385.pickle and b/code/data/sim_385.pickle differ
diff --git a/code/data/sim_386.pickle b/code/data/sim_386.pickle
index 2441191..7f18ef0 100644
Binary files a/code/data/sim_386.pickle and b/code/data/sim_386.pickle differ
diff --git a/code/data/sim_387.pickle b/code/data/sim_387.pickle
index cf3be22..9707beb 100644
Binary files a/code/data/sim_387.pickle and b/code/data/sim_387.pickle differ
diff --git a/code/data/sim_388.pickle b/code/data/sim_388.pickle
index 63fc475..a09a40b 100644
Binary files a/code/data/sim_388.pickle and b/code/data/sim_388.pickle differ
diff --git a/code/data/sim_389.pickle b/code/data/sim_389.pickle
index 85a226e..5d57b8e 100644
Binary files a/code/data/sim_389.pickle and b/code/data/sim_389.pickle differ
diff --git a/code/data/sim_39.pickle b/code/data/sim_39.pickle
index bab1cf8..8977739 100644
Binary files a/code/data/sim_39.pickle and b/code/data/sim_39.pickle differ
diff --git a/code/data/sim_390.pickle b/code/data/sim_390.pickle
index 79d5c77..1311527 100644
Binary files a/code/data/sim_390.pickle and b/code/data/sim_390.pickle differ
diff --git a/code/data/sim_391.pickle b/code/data/sim_391.pickle
index 1b332fa..e71b8ad 100644
Binary files a/code/data/sim_391.pickle and b/code/data/sim_391.pickle differ
diff --git a/code/data/sim_392.pickle b/code/data/sim_392.pickle
index cd79ef2..8e34b37 100644
Binary files a/code/data/sim_392.pickle and b/code/data/sim_392.pickle differ
diff --git a/code/data/sim_393.pickle b/code/data/sim_393.pickle
index 9078c63..486524b 100644
Binary files a/code/data/sim_393.pickle and b/code/data/sim_393.pickle differ
diff --git a/code/data/sim_394.pickle b/code/data/sim_394.pickle
index 8d3aaf9..f5ab888 100644
Binary files a/code/data/sim_394.pickle and b/code/data/sim_394.pickle differ
diff --git a/code/data/sim_395.pickle b/code/data/sim_395.pickle
index 246205c..a1f5fbb 100644
Binary files a/code/data/sim_395.pickle and b/code/data/sim_395.pickle differ
diff --git a/code/data/sim_396.pickle b/code/data/sim_396.pickle
index e82c92f..884433b 100644
Binary files a/code/data/sim_396.pickle and b/code/data/sim_396.pickle differ
diff --git a/code/data/sim_397.pickle b/code/data/sim_397.pickle
index c08e9d9..3058d24 100644
Binary files a/code/data/sim_397.pickle and b/code/data/sim_397.pickle differ
diff --git a/code/data/sim_398.pickle b/code/data/sim_398.pickle
index 7c976f8..f3d502c 100644
Binary files a/code/data/sim_398.pickle and b/code/data/sim_398.pickle differ
diff --git a/code/data/sim_399.pickle b/code/data/sim_399.pickle
index def0af3..63aa4fd 100644
Binary files a/code/data/sim_399.pickle and b/code/data/sim_399.pickle differ
diff --git a/code/data/sim_4.pickle b/code/data/sim_4.pickle
index b65b828..0d5225d 100644
Binary files a/code/data/sim_4.pickle and b/code/data/sim_4.pickle differ
diff --git a/code/data/sim_40.pickle b/code/data/sim_40.pickle
index 70451be..82d3ce6 100644
Binary files a/code/data/sim_40.pickle and b/code/data/sim_40.pickle differ
diff --git a/code/data/sim_400.pickle b/code/data/sim_400.pickle
index 4a6cb99..9a07010 100644
Binary files a/code/data/sim_400.pickle and b/code/data/sim_400.pickle differ
diff --git a/code/data/sim_401.pickle b/code/data/sim_401.pickle
index 887ca5d..6063c37 100644
Binary files a/code/data/sim_401.pickle and b/code/data/sim_401.pickle differ
diff --git a/code/data/sim_402.pickle b/code/data/sim_402.pickle
index bc652b3..5c43ab9 100644
Binary files a/code/data/sim_402.pickle and b/code/data/sim_402.pickle differ
diff --git a/code/data/sim_403.pickle b/code/data/sim_403.pickle
index bd0b5cd..3ab90a1 100644
Binary files a/code/data/sim_403.pickle and b/code/data/sim_403.pickle differ
diff --git a/code/data/sim_404.pickle b/code/data/sim_404.pickle
index 01a5de4..d3922b9 100644
Binary files a/code/data/sim_404.pickle and b/code/data/sim_404.pickle differ
diff --git a/code/data/sim_405.pickle b/code/data/sim_405.pickle
index b78d075..9da91f6 100644
Binary files a/code/data/sim_405.pickle and b/code/data/sim_405.pickle differ
diff --git a/code/data/sim_406.pickle b/code/data/sim_406.pickle
index fb4d373..e71ea8d 100644
Binary files a/code/data/sim_406.pickle and b/code/data/sim_406.pickle differ
diff --git a/code/data/sim_407.pickle b/code/data/sim_407.pickle
index dda9792..628b54f 100644
Binary files a/code/data/sim_407.pickle and b/code/data/sim_407.pickle differ
diff --git a/code/data/sim_408.pickle b/code/data/sim_408.pickle
index 40db86c..b4b94b0 100644
Binary files a/code/data/sim_408.pickle and b/code/data/sim_408.pickle differ
diff --git a/code/data/sim_409.pickle b/code/data/sim_409.pickle
index 53ea278..54cbab9 100644
Binary files a/code/data/sim_409.pickle and b/code/data/sim_409.pickle differ
diff --git a/code/data/sim_41.pickle b/code/data/sim_41.pickle
index 0b9d58d..208fbb7 100644
Binary files a/code/data/sim_41.pickle and b/code/data/sim_41.pickle differ
diff --git a/code/data/sim_410.pickle b/code/data/sim_410.pickle
index 06a681a..7b4f33a 100644
Binary files a/code/data/sim_410.pickle and b/code/data/sim_410.pickle differ
diff --git a/code/data/sim_411.pickle b/code/data/sim_411.pickle
index 2b9d96c..4ed1a69 100644
Binary files a/code/data/sim_411.pickle and b/code/data/sim_411.pickle differ
diff --git a/code/data/sim_412.pickle b/code/data/sim_412.pickle
index 90d38d1..2c08b4e 100644
Binary files a/code/data/sim_412.pickle and b/code/data/sim_412.pickle differ
diff --git a/code/data/sim_413.pickle b/code/data/sim_413.pickle
index c82a0b3..1277ff1 100644
Binary files a/code/data/sim_413.pickle and b/code/data/sim_413.pickle differ
diff --git a/code/data/sim_414.pickle b/code/data/sim_414.pickle
index 1ed57fd..ce37d5f 100644
Binary files a/code/data/sim_414.pickle and b/code/data/sim_414.pickle differ
diff --git a/code/data/sim_415.pickle b/code/data/sim_415.pickle
index 456d992..b052d1b 100644
Binary files a/code/data/sim_415.pickle and b/code/data/sim_415.pickle differ
diff --git a/code/data/sim_416.pickle b/code/data/sim_416.pickle
index 4d2adb2..ac28f4f 100644
Binary files a/code/data/sim_416.pickle and b/code/data/sim_416.pickle differ
diff --git a/code/data/sim_417.pickle b/code/data/sim_417.pickle
index 428f037..1ae2fb2 100644
Binary files a/code/data/sim_417.pickle and b/code/data/sim_417.pickle differ
diff --git a/code/data/sim_418.pickle b/code/data/sim_418.pickle
index f5765c5..2548f0a 100644
Binary files a/code/data/sim_418.pickle and b/code/data/sim_418.pickle differ
diff --git a/code/data/sim_419.pickle b/code/data/sim_419.pickle
index bd33ed0..b927e09 100644
Binary files a/code/data/sim_419.pickle and b/code/data/sim_419.pickle differ
diff --git a/code/data/sim_42.pickle b/code/data/sim_42.pickle
index a608c46..0d50841 100644
Binary files a/code/data/sim_42.pickle and b/code/data/sim_42.pickle differ
diff --git a/code/data/sim_420.pickle b/code/data/sim_420.pickle
index 8b26e5b..a01c9e3 100644
Binary files a/code/data/sim_420.pickle and b/code/data/sim_420.pickle differ
diff --git a/code/data/sim_421.pickle b/code/data/sim_421.pickle
index 91a9090..e199bda 100644
Binary files a/code/data/sim_421.pickle and b/code/data/sim_421.pickle differ
diff --git a/code/data/sim_422.pickle b/code/data/sim_422.pickle
index 5bd6c4a..51997ff 100644
Binary files a/code/data/sim_422.pickle and b/code/data/sim_422.pickle differ
diff --git a/code/data/sim_423.pickle b/code/data/sim_423.pickle
index 58b5d92..5ddeb69 100644
Binary files a/code/data/sim_423.pickle and b/code/data/sim_423.pickle differ
diff --git a/code/data/sim_424.pickle b/code/data/sim_424.pickle
index 45ac7d2..c89276a 100644
Binary files a/code/data/sim_424.pickle and b/code/data/sim_424.pickle differ
diff --git a/code/data/sim_425.pickle b/code/data/sim_425.pickle
index 73d3b70..1df15f2 100644
Binary files a/code/data/sim_425.pickle and b/code/data/sim_425.pickle differ
diff --git a/code/data/sim_426.pickle b/code/data/sim_426.pickle
index 6d9694b..d7fc344 100644
Binary files a/code/data/sim_426.pickle and b/code/data/sim_426.pickle differ
diff --git a/code/data/sim_427.pickle b/code/data/sim_427.pickle
index 1dd23ba..d266631 100644
Binary files a/code/data/sim_427.pickle and b/code/data/sim_427.pickle differ
diff --git a/code/data/sim_428.pickle b/code/data/sim_428.pickle
index 6307332..b59bd63 100644
Binary files a/code/data/sim_428.pickle and b/code/data/sim_428.pickle differ
diff --git a/code/data/sim_429.pickle b/code/data/sim_429.pickle
index 2fa18bb..aa7f72c 100644
Binary files a/code/data/sim_429.pickle and b/code/data/sim_429.pickle differ
diff --git a/code/data/sim_43.pickle b/code/data/sim_43.pickle
index e09cadb..4fe584e 100644
Binary files a/code/data/sim_43.pickle and b/code/data/sim_43.pickle differ
diff --git a/code/data/sim_430.pickle b/code/data/sim_430.pickle
index 1e8c2cc..203b4c7 100644
Binary files a/code/data/sim_430.pickle and b/code/data/sim_430.pickle differ
diff --git a/code/data/sim_431.pickle b/code/data/sim_431.pickle
index 5faf791..752e5dc 100644
Binary files a/code/data/sim_431.pickle and b/code/data/sim_431.pickle differ
diff --git a/code/data/sim_432.pickle b/code/data/sim_432.pickle
index 83f37ef..bc82608 100644
Binary files a/code/data/sim_432.pickle and b/code/data/sim_432.pickle differ
diff --git a/code/data/sim_433.pickle b/code/data/sim_433.pickle
index d8d6153..f66d4eb 100644
Binary files a/code/data/sim_433.pickle and b/code/data/sim_433.pickle differ
diff --git a/code/data/sim_434.pickle b/code/data/sim_434.pickle
index 032bdff..3d902f4 100644
Binary files a/code/data/sim_434.pickle and b/code/data/sim_434.pickle differ
diff --git a/code/data/sim_435.pickle b/code/data/sim_435.pickle
index 4db8d0a..c3ea2b9 100644
Binary files a/code/data/sim_435.pickle and b/code/data/sim_435.pickle differ
diff --git a/code/data/sim_436.pickle b/code/data/sim_436.pickle
index 32eb553..254b067 100644
Binary files a/code/data/sim_436.pickle and b/code/data/sim_436.pickle differ
diff --git a/code/data/sim_437.pickle b/code/data/sim_437.pickle
index 8b8c593..2a0709c 100644
Binary files a/code/data/sim_437.pickle and b/code/data/sim_437.pickle differ
diff --git a/code/data/sim_438.pickle b/code/data/sim_438.pickle
index ea39495..6a68280 100644
Binary files a/code/data/sim_438.pickle and b/code/data/sim_438.pickle differ
diff --git a/code/data/sim_439.pickle b/code/data/sim_439.pickle
index 994521c..5ddbdf3 100644
Binary files a/code/data/sim_439.pickle and b/code/data/sim_439.pickle differ
diff --git a/code/data/sim_44.pickle b/code/data/sim_44.pickle
index bd6d6b0..280ce59 100644
Binary files a/code/data/sim_44.pickle and b/code/data/sim_44.pickle differ
diff --git a/code/data/sim_440.pickle b/code/data/sim_440.pickle
index 79c6e87..ba68c5c 100644
Binary files a/code/data/sim_440.pickle and b/code/data/sim_440.pickle differ
diff --git a/code/data/sim_441.pickle b/code/data/sim_441.pickle
index d418291..9eef336 100644
Binary files a/code/data/sim_441.pickle and b/code/data/sim_441.pickle differ
diff --git a/code/data/sim_442.pickle b/code/data/sim_442.pickle
index 21d2624..1ed5c88 100644
Binary files a/code/data/sim_442.pickle and b/code/data/sim_442.pickle differ
diff --git a/code/data/sim_443.pickle b/code/data/sim_443.pickle
index e83c9b4..cbb02e6 100644
Binary files a/code/data/sim_443.pickle and b/code/data/sim_443.pickle differ
diff --git a/code/data/sim_444.pickle b/code/data/sim_444.pickle
index 3c403ab..7f81bd2 100644
Binary files a/code/data/sim_444.pickle and b/code/data/sim_444.pickle differ
diff --git a/code/data/sim_445.pickle b/code/data/sim_445.pickle
index f9b6a53..86f8050 100644
Binary files a/code/data/sim_445.pickle and b/code/data/sim_445.pickle differ
diff --git a/code/data/sim_446.pickle b/code/data/sim_446.pickle
index a5224c6..555454a 100644
Binary files a/code/data/sim_446.pickle and b/code/data/sim_446.pickle differ
diff --git a/code/data/sim_447.pickle b/code/data/sim_447.pickle
index 8c5b08b..cb62dd1 100644
Binary files a/code/data/sim_447.pickle and b/code/data/sim_447.pickle differ
diff --git a/code/data/sim_448.pickle b/code/data/sim_448.pickle
index 2e2c07b..65cf2b1 100644
Binary files a/code/data/sim_448.pickle and b/code/data/sim_448.pickle differ
diff --git a/code/data/sim_449.pickle b/code/data/sim_449.pickle
index 486276e..0dbb117 100644
Binary files a/code/data/sim_449.pickle and b/code/data/sim_449.pickle differ
diff --git a/code/data/sim_45.pickle b/code/data/sim_45.pickle
index e775128..1f44d72 100644
Binary files a/code/data/sim_45.pickle and b/code/data/sim_45.pickle differ
diff --git a/code/data/sim_450.pickle b/code/data/sim_450.pickle
index 354d4d3..df60af9 100644
Binary files a/code/data/sim_450.pickle and b/code/data/sim_450.pickle differ
diff --git a/code/data/sim_451.pickle b/code/data/sim_451.pickle
index bbf21b9..9dc7737 100644
Binary files a/code/data/sim_451.pickle and b/code/data/sim_451.pickle differ
diff --git a/code/data/sim_452.pickle b/code/data/sim_452.pickle
index 971948e..2bbf7c6 100644
Binary files a/code/data/sim_452.pickle and b/code/data/sim_452.pickle differ
diff --git a/code/data/sim_453.pickle b/code/data/sim_453.pickle
index e1e4249..59c04f5 100644
Binary files a/code/data/sim_453.pickle and b/code/data/sim_453.pickle differ
diff --git a/code/data/sim_454.pickle b/code/data/sim_454.pickle
index 7bc6a32..1267cd0 100644
Binary files a/code/data/sim_454.pickle and b/code/data/sim_454.pickle differ
diff --git a/code/data/sim_455.pickle b/code/data/sim_455.pickle
index 1293bff..2b30200 100644
Binary files a/code/data/sim_455.pickle and b/code/data/sim_455.pickle differ
diff --git a/code/data/sim_456.pickle b/code/data/sim_456.pickle
index 9152511..1653e25 100644
Binary files a/code/data/sim_456.pickle and b/code/data/sim_456.pickle differ
diff --git a/code/data/sim_457.pickle b/code/data/sim_457.pickle
index c41d83f..ab0f113 100644
Binary files a/code/data/sim_457.pickle and b/code/data/sim_457.pickle differ
diff --git a/code/data/sim_458.pickle b/code/data/sim_458.pickle
index cdf423e..a836534 100644
Binary files a/code/data/sim_458.pickle and b/code/data/sim_458.pickle differ
diff --git a/code/data/sim_459.pickle b/code/data/sim_459.pickle
index 4e63351..95ac380 100644
Binary files a/code/data/sim_459.pickle and b/code/data/sim_459.pickle differ
diff --git a/code/data/sim_46.pickle b/code/data/sim_46.pickle
index 43f9c69..f767a93 100644
Binary files a/code/data/sim_46.pickle and b/code/data/sim_46.pickle differ
diff --git a/code/data/sim_460.pickle b/code/data/sim_460.pickle
index d3b8402..a69de6d 100644
Binary files a/code/data/sim_460.pickle and b/code/data/sim_460.pickle differ
diff --git a/code/data/sim_461.pickle b/code/data/sim_461.pickle
index 25aae39..1a4d93b 100644
Binary files a/code/data/sim_461.pickle and b/code/data/sim_461.pickle differ
diff --git a/code/data/sim_462.pickle b/code/data/sim_462.pickle
index 3b38c14..a124a23 100644
Binary files a/code/data/sim_462.pickle and b/code/data/sim_462.pickle differ
diff --git a/code/data/sim_463.pickle b/code/data/sim_463.pickle
index 0592fa2..c811df5 100644
Binary files a/code/data/sim_463.pickle and b/code/data/sim_463.pickle differ
diff --git a/code/data/sim_464.pickle b/code/data/sim_464.pickle
index c06ede9..e168208 100644
Binary files a/code/data/sim_464.pickle and b/code/data/sim_464.pickle differ
diff --git a/code/data/sim_465.pickle b/code/data/sim_465.pickle
index 4ef05de..057d03d 100644
Binary files a/code/data/sim_465.pickle and b/code/data/sim_465.pickle differ
diff --git a/code/data/sim_466.pickle b/code/data/sim_466.pickle
index 37677b5..86de069 100644
Binary files a/code/data/sim_466.pickle and b/code/data/sim_466.pickle differ
diff --git a/code/data/sim_467.pickle b/code/data/sim_467.pickle
index 1367018..230c1e3 100644
Binary files a/code/data/sim_467.pickle and b/code/data/sim_467.pickle differ
diff --git a/code/data/sim_468.pickle b/code/data/sim_468.pickle
index 0eb3f73..b34737e 100644
Binary files a/code/data/sim_468.pickle and b/code/data/sim_468.pickle differ
diff --git a/code/data/sim_469.pickle b/code/data/sim_469.pickle
index 02f66c2..7707789 100644
Binary files a/code/data/sim_469.pickle and b/code/data/sim_469.pickle differ
diff --git a/code/data/sim_47.pickle b/code/data/sim_47.pickle
index 9135b1a..ff4e7b9 100644
Binary files a/code/data/sim_47.pickle and b/code/data/sim_47.pickle differ
diff --git a/code/data/sim_470.pickle b/code/data/sim_470.pickle
index 869dc86..4aeb29b 100644
Binary files a/code/data/sim_470.pickle and b/code/data/sim_470.pickle differ
diff --git a/code/data/sim_471.pickle b/code/data/sim_471.pickle
index 308659f..375aa8a 100644
Binary files a/code/data/sim_471.pickle and b/code/data/sim_471.pickle differ
diff --git a/code/data/sim_472.pickle b/code/data/sim_472.pickle
index 7a94388..c8261e4 100644
Binary files a/code/data/sim_472.pickle and b/code/data/sim_472.pickle differ
diff --git a/code/data/sim_473.pickle b/code/data/sim_473.pickle
index 2fd768b..804f97a 100644
Binary files a/code/data/sim_473.pickle and b/code/data/sim_473.pickle differ
diff --git a/code/data/sim_474.pickle b/code/data/sim_474.pickle
index 9144ce7..925ce41 100644
Binary files a/code/data/sim_474.pickle and b/code/data/sim_474.pickle differ
diff --git a/code/data/sim_475.pickle b/code/data/sim_475.pickle
index 6ba22e9..2530b2c 100644
Binary files a/code/data/sim_475.pickle and b/code/data/sim_475.pickle differ
diff --git a/code/data/sim_476.pickle b/code/data/sim_476.pickle
index c2a3e24..ecb7b6b 100644
Binary files a/code/data/sim_476.pickle and b/code/data/sim_476.pickle differ
diff --git a/code/data/sim_477.pickle b/code/data/sim_477.pickle
index 7580ca3..eb35c9e 100644
Binary files a/code/data/sim_477.pickle and b/code/data/sim_477.pickle differ
diff --git a/code/data/sim_478.pickle b/code/data/sim_478.pickle
index 377aa41..5e7615d 100644
Binary files a/code/data/sim_478.pickle and b/code/data/sim_478.pickle differ
diff --git a/code/data/sim_479.pickle b/code/data/sim_479.pickle
index 718ea5b..6487806 100644
Binary files a/code/data/sim_479.pickle and b/code/data/sim_479.pickle differ
diff --git a/code/data/sim_48.pickle b/code/data/sim_48.pickle
index e2aad6c..b7dad69 100644
Binary files a/code/data/sim_48.pickle and b/code/data/sim_48.pickle differ
diff --git a/code/data/sim_480.pickle b/code/data/sim_480.pickle
index 54cbf31..fc37b00 100644
Binary files a/code/data/sim_480.pickle and b/code/data/sim_480.pickle differ
diff --git a/code/data/sim_481.pickle b/code/data/sim_481.pickle
index 451dd6b..316b8ac 100644
Binary files a/code/data/sim_481.pickle and b/code/data/sim_481.pickle differ
diff --git a/code/data/sim_482.pickle b/code/data/sim_482.pickle
index b89e53d..dbf47b9 100644
Binary files a/code/data/sim_482.pickle and b/code/data/sim_482.pickle differ
diff --git a/code/data/sim_483.pickle b/code/data/sim_483.pickle
index 639144f..bb99779 100644
Binary files a/code/data/sim_483.pickle and b/code/data/sim_483.pickle differ
diff --git a/code/data/sim_484.pickle b/code/data/sim_484.pickle
index 48bb123..43d23d0 100644
Binary files a/code/data/sim_484.pickle and b/code/data/sim_484.pickle differ
diff --git a/code/data/sim_485.pickle b/code/data/sim_485.pickle
index 4013f93..058fe50 100644
Binary files a/code/data/sim_485.pickle and b/code/data/sim_485.pickle differ
diff --git a/code/data/sim_486.pickle b/code/data/sim_486.pickle
index d68e0ea..daee73a 100644
Binary files a/code/data/sim_486.pickle and b/code/data/sim_486.pickle differ
diff --git a/code/data/sim_487.pickle b/code/data/sim_487.pickle
index 31c2cbc..8677302 100644
Binary files a/code/data/sim_487.pickle and b/code/data/sim_487.pickle differ
diff --git a/code/data/sim_488.pickle b/code/data/sim_488.pickle
index bff76c5..a037c06 100644
Binary files a/code/data/sim_488.pickle and b/code/data/sim_488.pickle differ
diff --git a/code/data/sim_489.pickle b/code/data/sim_489.pickle
index 5519edd..5456ab4 100644
Binary files a/code/data/sim_489.pickle and b/code/data/sim_489.pickle differ
diff --git a/code/data/sim_49.pickle b/code/data/sim_49.pickle
index d8c22f3..5107912 100644
Binary files a/code/data/sim_49.pickle and b/code/data/sim_49.pickle differ
diff --git a/code/data/sim_490.pickle b/code/data/sim_490.pickle
index e37590b..0b1c9b3 100644
Binary files a/code/data/sim_490.pickle and b/code/data/sim_490.pickle differ
diff --git a/code/data/sim_491.pickle b/code/data/sim_491.pickle
index ef27e6d..a1c6f08 100644
Binary files a/code/data/sim_491.pickle and b/code/data/sim_491.pickle differ
diff --git a/code/data/sim_492.pickle b/code/data/sim_492.pickle
index 099e2c7..1c8d2f6 100644
Binary files a/code/data/sim_492.pickle and b/code/data/sim_492.pickle differ
diff --git a/code/data/sim_493.pickle b/code/data/sim_493.pickle
index 1cf0e1f..bbbaa4e 100644
Binary files a/code/data/sim_493.pickle and b/code/data/sim_493.pickle differ
diff --git a/code/data/sim_494.pickle b/code/data/sim_494.pickle
index 4c7d4f1..ee58855 100644
Binary files a/code/data/sim_494.pickle and b/code/data/sim_494.pickle differ
diff --git a/code/data/sim_495.pickle b/code/data/sim_495.pickle
index 5160ca1..daf3c78 100644
Binary files a/code/data/sim_495.pickle and b/code/data/sim_495.pickle differ
diff --git a/code/data/sim_496.pickle b/code/data/sim_496.pickle
index e7d9170..e44f499 100644
Binary files a/code/data/sim_496.pickle and b/code/data/sim_496.pickle differ
diff --git a/code/data/sim_497.pickle b/code/data/sim_497.pickle
index 775b87a..ca5e9b0 100644
Binary files a/code/data/sim_497.pickle and b/code/data/sim_497.pickle differ
diff --git a/code/data/sim_498.pickle b/code/data/sim_498.pickle
index f116b37..eccc356 100644
Binary files a/code/data/sim_498.pickle and b/code/data/sim_498.pickle differ
diff --git a/code/data/sim_499.pickle b/code/data/sim_499.pickle
index e539c96..9b58fa9 100644
Binary files a/code/data/sim_499.pickle and b/code/data/sim_499.pickle differ
diff --git a/code/data/sim_5.pickle b/code/data/sim_5.pickle
index f02d894..1b6433f 100644
Binary files a/code/data/sim_5.pickle and b/code/data/sim_5.pickle differ
diff --git a/code/data/sim_50.pickle b/code/data/sim_50.pickle
index cf5adb3..e70c008 100644
Binary files a/code/data/sim_50.pickle and b/code/data/sim_50.pickle differ
diff --git a/code/data/sim_51.pickle b/code/data/sim_51.pickle
index c64a058..0632ef4 100644
Binary files a/code/data/sim_51.pickle and b/code/data/sim_51.pickle differ
diff --git a/code/data/sim_52.pickle b/code/data/sim_52.pickle
index a9bb7d2..2c064f0 100644
Binary files a/code/data/sim_52.pickle and b/code/data/sim_52.pickle differ
diff --git a/code/data/sim_53.pickle b/code/data/sim_53.pickle
index be451d0..3ed7f9f 100644
Binary files a/code/data/sim_53.pickle and b/code/data/sim_53.pickle differ
diff --git a/code/data/sim_54.pickle b/code/data/sim_54.pickle
index cb625d8..ae49f00 100644
Binary files a/code/data/sim_54.pickle and b/code/data/sim_54.pickle differ
diff --git a/code/data/sim_55.pickle b/code/data/sim_55.pickle
index 8659792..4a52f7b 100644
Binary files a/code/data/sim_55.pickle and b/code/data/sim_55.pickle differ
diff --git a/code/data/sim_56.pickle b/code/data/sim_56.pickle
index f0a6d50..3c9cb09 100644
Binary files a/code/data/sim_56.pickle and b/code/data/sim_56.pickle differ
diff --git a/code/data/sim_57.pickle b/code/data/sim_57.pickle
index f64caa0..fa9561a 100644
Binary files a/code/data/sim_57.pickle and b/code/data/sim_57.pickle differ
diff --git a/code/data/sim_58.pickle b/code/data/sim_58.pickle
index 30f8002..e7f933a 100644
Binary files a/code/data/sim_58.pickle and b/code/data/sim_58.pickle differ
diff --git a/code/data/sim_59.pickle b/code/data/sim_59.pickle
index 10df099..d954f14 100644
Binary files a/code/data/sim_59.pickle and b/code/data/sim_59.pickle differ
diff --git a/code/data/sim_6.pickle b/code/data/sim_6.pickle
index 659ce32..7183e24 100644
Binary files a/code/data/sim_6.pickle and b/code/data/sim_6.pickle differ
diff --git a/code/data/sim_60.pickle b/code/data/sim_60.pickle
index 87a0c15..bdb01b1 100644
Binary files a/code/data/sim_60.pickle and b/code/data/sim_60.pickle differ
diff --git a/code/data/sim_61.pickle b/code/data/sim_61.pickle
index e9eb070..ae2f667 100644
Binary files a/code/data/sim_61.pickle and b/code/data/sim_61.pickle differ
diff --git a/code/data/sim_62.pickle b/code/data/sim_62.pickle
index 2e79c8f..faa19d7 100644
Binary files a/code/data/sim_62.pickle and b/code/data/sim_62.pickle differ
diff --git a/code/data/sim_63.pickle b/code/data/sim_63.pickle
index 24f8d68..6d7161c 100644
Binary files a/code/data/sim_63.pickle and b/code/data/sim_63.pickle differ
diff --git a/code/data/sim_64.pickle b/code/data/sim_64.pickle
index a516175..7dff451 100644
Binary files a/code/data/sim_64.pickle and b/code/data/sim_64.pickle differ
diff --git a/code/data/sim_65.pickle b/code/data/sim_65.pickle
index f913828..1346ed4 100644
Binary files a/code/data/sim_65.pickle and b/code/data/sim_65.pickle differ
diff --git a/code/data/sim_66.pickle b/code/data/sim_66.pickle
index 8b68a4a..fd8f889 100644
Binary files a/code/data/sim_66.pickle and b/code/data/sim_66.pickle differ
diff --git a/code/data/sim_67.pickle b/code/data/sim_67.pickle
index 1b122e6..d5e04e1 100644
Binary files a/code/data/sim_67.pickle and b/code/data/sim_67.pickle differ
diff --git a/code/data/sim_68.pickle b/code/data/sim_68.pickle
index 3659fcc..d7cacfa 100644
Binary files a/code/data/sim_68.pickle and b/code/data/sim_68.pickle differ
diff --git a/code/data/sim_69.pickle b/code/data/sim_69.pickle
index 576f2ee..71fd255 100644
Binary files a/code/data/sim_69.pickle and b/code/data/sim_69.pickle differ
diff --git a/code/data/sim_7.pickle b/code/data/sim_7.pickle
index e4cb550..b6b6ae0 100644
Binary files a/code/data/sim_7.pickle and b/code/data/sim_7.pickle differ
diff --git a/code/data/sim_70.pickle b/code/data/sim_70.pickle
index 699a080..d3ee683 100644
Binary files a/code/data/sim_70.pickle and b/code/data/sim_70.pickle differ
diff --git a/code/data/sim_71.pickle b/code/data/sim_71.pickle
index 2c9e894..f38bfc6 100644
Binary files a/code/data/sim_71.pickle and b/code/data/sim_71.pickle differ
diff --git a/code/data/sim_72.pickle b/code/data/sim_72.pickle
index 50c539b..7d20274 100644
Binary files a/code/data/sim_72.pickle and b/code/data/sim_72.pickle differ
diff --git a/code/data/sim_73.pickle b/code/data/sim_73.pickle
index d176872..393c45f 100644
Binary files a/code/data/sim_73.pickle and b/code/data/sim_73.pickle differ
diff --git a/code/data/sim_74.pickle b/code/data/sim_74.pickle
index bc116f3..e4047a5 100644
Binary files a/code/data/sim_74.pickle and b/code/data/sim_74.pickle differ
diff --git a/code/data/sim_75.pickle b/code/data/sim_75.pickle
index 7871679..427201b 100644
Binary files a/code/data/sim_75.pickle and b/code/data/sim_75.pickle differ
diff --git a/code/data/sim_76.pickle b/code/data/sim_76.pickle
index 6eb5b73..8457f12 100644
Binary files a/code/data/sim_76.pickle and b/code/data/sim_76.pickle differ
diff --git a/code/data/sim_77.pickle b/code/data/sim_77.pickle
index b73f0dd..fd9e5ab 100644
Binary files a/code/data/sim_77.pickle and b/code/data/sim_77.pickle differ
diff --git a/code/data/sim_78.pickle b/code/data/sim_78.pickle
index e77667a..4e35982 100644
Binary files a/code/data/sim_78.pickle and b/code/data/sim_78.pickle differ
diff --git a/code/data/sim_79.pickle b/code/data/sim_79.pickle
index 910d7da..7df30a3 100644
Binary files a/code/data/sim_79.pickle and b/code/data/sim_79.pickle differ
diff --git a/code/data/sim_8.pickle b/code/data/sim_8.pickle
index f93d7f8..d78528a 100644
Binary files a/code/data/sim_8.pickle and b/code/data/sim_8.pickle differ
diff --git a/code/data/sim_80.pickle b/code/data/sim_80.pickle
index 5e9edac..55b463e 100644
Binary files a/code/data/sim_80.pickle and b/code/data/sim_80.pickle differ
diff --git a/code/data/sim_81.pickle b/code/data/sim_81.pickle
index 3f51e5b..e7ca058 100644
Binary files a/code/data/sim_81.pickle and b/code/data/sim_81.pickle differ
diff --git a/code/data/sim_82.pickle b/code/data/sim_82.pickle
index ef848d9..3d84935 100644
Binary files a/code/data/sim_82.pickle and b/code/data/sim_82.pickle differ
diff --git a/code/data/sim_83.pickle b/code/data/sim_83.pickle
index c964918..77ed78b 100644
Binary files a/code/data/sim_83.pickle and b/code/data/sim_83.pickle differ
diff --git a/code/data/sim_84.pickle b/code/data/sim_84.pickle
index bda7be0..1b788b8 100644
Binary files a/code/data/sim_84.pickle and b/code/data/sim_84.pickle differ
diff --git a/code/data/sim_85.pickle b/code/data/sim_85.pickle
index cce4473..b0b608a 100644
Binary files a/code/data/sim_85.pickle and b/code/data/sim_85.pickle differ
diff --git a/code/data/sim_86.pickle b/code/data/sim_86.pickle
index d28ea36..3ffce36 100644
Binary files a/code/data/sim_86.pickle and b/code/data/sim_86.pickle differ
diff --git a/code/data/sim_87.pickle b/code/data/sim_87.pickle
index da578b6..1fef439 100644
Binary files a/code/data/sim_87.pickle and b/code/data/sim_87.pickle differ
diff --git a/code/data/sim_88.pickle b/code/data/sim_88.pickle
index f8f6354..f967f6a 100644
Binary files a/code/data/sim_88.pickle and b/code/data/sim_88.pickle differ
diff --git a/code/data/sim_89.pickle b/code/data/sim_89.pickle
index 024f528..b21f061 100644
Binary files a/code/data/sim_89.pickle and b/code/data/sim_89.pickle differ
diff --git a/code/data/sim_9.pickle b/code/data/sim_9.pickle
index 1508a7e..09a9a91 100644
Binary files a/code/data/sim_9.pickle and b/code/data/sim_9.pickle differ
diff --git a/code/data/sim_90.pickle b/code/data/sim_90.pickle
index 85f90a0..8226983 100644
Binary files a/code/data/sim_90.pickle and b/code/data/sim_90.pickle differ
diff --git a/code/data/sim_91.pickle b/code/data/sim_91.pickle
index 560ea10..e3c887a 100644
Binary files a/code/data/sim_91.pickle and b/code/data/sim_91.pickle differ
diff --git a/code/data/sim_92.pickle b/code/data/sim_92.pickle
index eb10737..648560b 100644
Binary files a/code/data/sim_92.pickle and b/code/data/sim_92.pickle differ
diff --git a/code/data/sim_93.pickle b/code/data/sim_93.pickle
index 718fa5d..3c2e6cb 100644
Binary files a/code/data/sim_93.pickle and b/code/data/sim_93.pickle differ
diff --git a/code/data/sim_94.pickle b/code/data/sim_94.pickle
index 8954cb7..30e411b 100644
Binary files a/code/data/sim_94.pickle and b/code/data/sim_94.pickle differ
diff --git a/code/data/sim_95.pickle b/code/data/sim_95.pickle
index 217cf24..5700f96 100644
Binary files a/code/data/sim_95.pickle and b/code/data/sim_95.pickle differ
diff --git a/code/data/sim_96.pickle b/code/data/sim_96.pickle
index 2e636ce..5bf59df 100644
Binary files a/code/data/sim_96.pickle and b/code/data/sim_96.pickle differ
diff --git a/code/data/sim_97.pickle b/code/data/sim_97.pickle
index 0f2fde1..ede338d 100644
Binary files a/code/data/sim_97.pickle and b/code/data/sim_97.pickle differ
diff --git a/code/data/sim_98.pickle b/code/data/sim_98.pickle
index 67ecd01..a0cf254 100644
Binary files a/code/data/sim_98.pickle and b/code/data/sim_98.pickle differ
diff --git a/code/data/sim_99.pickle b/code/data/sim_99.pickle
index ab6a534..60f837e 100644
Binary files a/code/data/sim_99.pickle and b/code/data/sim_99.pickle differ
diff --git a/code/wandb/latest-run b/code/wandb/latest-run
index 9e5d79d..40a0521 120000
--- a/code/wandb/latest-run
+++ b/code/wandb/latest-run
@@ -1 +1 @@
-run-20220913_145719-8chtqm30
\ No newline at end of file
+run-20220913_164802-33zipr3z
\ No newline at end of file
